
 backend-2 Pulling 
 backend-1 Pulling 
 9824c27679d3 Already exists 
 1a283093615e Already exists 
 c0055458ac5d Already exists 
 b7ce9d4e80c0 Pulling fs layer 
 62aabecb7dc8 Pulling fs layer 
 9824c27679d3 Already exists 
 1a283093615e Already exists 
 c0055458ac5d Already exists 
 b7ce9d4e80c0 Pulling fs layer 
 62aabecb7dc8 Pulling fs layer 
 62aabecb7dc8 Downloading [>                                                  ]  228.2kB/22.61MB
 62aabecb7dc8 Downloading [>                                                  ]  228.2kB/22.61MB
 62aabecb7dc8 Downloading [==================>                                ]  8.487MB/22.61MB
 62aabecb7dc8 Downloading [==================>                                ]  8.487MB/22.61MB
 b7ce9d4e80c0 Downloading [==================================================>]      93B/93B
 b7ce9d4e80c0 Verifying Checksum 
 b7ce9d4e80c0 Download complete 
 b7ce9d4e80c0 Downloading [==================================================>]      93B/93B
 b7ce9d4e80c0 Verifying Checksum 
 b7ce9d4e80c0 Download complete 
 b7ce9d4e80c0 Extracting [==================================================>]      93B/93B
 b7ce9d4e80c0 Extracting [==================================================>]      93B/93B
 b7ce9d4e80c0 Extracting [==================================================>]      93B/93B
 b7ce9d4e80c0 Extracting [==================================================>]      93B/93B
 b7ce9d4e80c0 Pull complete 
 b7ce9d4e80c0 Pull complete 
 62aabecb7dc8 Download complete 
 62aabecb7dc8 Download complete 
 62aabecb7dc8 Extracting [>                                                  ]  229.4kB/22.61MB
 62aabecb7dc8 Extracting [>                                                  ]  229.4kB/22.61MB
 62aabecb7dc8 Extracting [====================>                              ]  9.404MB/22.61MB
 62aabecb7dc8 Extracting [====================>                              ]  9.404MB/22.61MB
 62aabecb7dc8 Extracting [=============================================>     ]  20.41MB/22.61MB
 62aabecb7dc8 Extracting [=============================================>     ]  20.41MB/22.61MB
 62aabecb7dc8 Extracting [==================================================>]  22.61MB/22.61MB
 62aabecb7dc8 Extracting [==================================================>]  22.61MB/22.61MB
 62aabecb7dc8 Pull complete 
 62aabecb7dc8 Pull complete 
 backend-1 Pulled 
 backend-2 Pulled 
 Network micheloliveira-com-dotnet-csharp-redis-grpc_backend  Creating
 Network micheloliveira-com-dotnet-csharp-redis-grpc_backend  Created
 Container micheloliveira-com-dotnet-csharp-redis-grpc-backend-2-1  Creating
 Container micheloliveira-com-dotnet-csharp-redis-grpc-redis-1  Creating
 Container micheloliveira-com-dotnet-csharp-redis-grpc-backend-1-1  Creating
 Container micheloliveira-com-dotnet-csharp-redis-grpc-backend-2-1  Created
 Container micheloliveira-com-dotnet-csharp-redis-grpc-backend-1-1  Created
 Container micheloliveira-com-dotnet-csharp-redis-grpc-load-balancer-1  Creating
 Container micheloliveira-com-dotnet-csharp-redis-grpc-redis-1  Created
 Container micheloliveira-com-dotnet-csharp-redis-grpc-load-balancer-1  Created
Attaching to backend-1-1, backend-2-1, load-balancer-1, redis-1
backend-1-1  | WORKER_SIZE: 15
backend-1-1  | BATCH_SIZE: 100
backend-1-1  | [Redis] Attempting connection...
backend-1-1  | Retry 1: RpcException - Status(StatusCode="Unavailable", Detail="Error connecting to subchannel.", DebugException="System.Net.Sockets.SocketException: Connection refused")
backend-1-1  | StackTrace:
backend-1-1  |    at Grpc.Net.Client.Balancer.Internal.ConnectionManager.<PickAsync>d__46.MoveNext() + 0x43f
backend-1-1  | --- End of stack trace from previous location ---
backend-1-1  |    at Grpc.Net.Client.Balancer.Internal.BalancerHttpHandler.<SendAsync>d__11.MoveNext() + 0x371
backend-1-1  | --- End of stack trace from previous location ---
backend-1-1  |    at Grpc.Net.Client.Internal.GrpcCall`2.<RunCall>d__82.MoveNext() + 0xefd
backend-1-1  | --- End of stack trace from previous location ---
backend-1-1  |    at MichelOliveira.Com.ReactiveLock.Distributed.Grpc.ReactiveLockGrpcTrackerStore.<SetStatusAsync>d__4.MoveNext() + 0x7a
backend-1-1  | --- End of stack trace from previous location ---
backend-1-1  |    at MichelOliveira.Com.ReactiveLock.Core.ReactiveLockTrackerController.<DecrementAsync>d__10.MoveNext() + 0x68
backend-1-1  | --- End of stack trace from previous location ---
backend-1-1  |    at MichelOliveira.Com.ReactiveLock.Distributed.Grpc.ReactiveLockGrpcTrackerExtensions.<UseDistributedGrpcReactiveLockAsync>d__10.MoveNext() + 0x2ac
backend-1-1  | --- End of stack trace from previous location ---
backend-1-1  |    at Program.<>c__DisplayClass0_0.<<<Main>$>b__21>d.MoveNext() + 0x50
backend-1-1  | --- End of stack trace from previous location ---
backend-1-1  |    at Polly.AsyncPolicy.<>c__DisplayClass40_0.<<ImplementationAsync>b__0>d.MoveNext() + 0x68
backend-1-1  | --- End of stack trace from previous location ---
backend-1-1  |    at Polly.Retry.AsyncRetryEngine.<ImplementationAsync>d__0`1.MoveNext() + 0x1c5
backend-1-1  | ----------------------------------------
backend-1-1  | Retry 1: RedisConnectionException - It was not possible to connect to the redis server(s). Error connecting right now. To allow this multiplexer to continue retrying until it's able to connect, use abortConnect=false in your connection string or AbortOnConnectFail=false; in your code.
backend-1-1  | StackTrace:
backend-1-1  |    at StackExchange.Redis.ConnectionMultiplexer.ConnectImpl(ConfigurationOptions, TextWriter, Nullable`1, EndPointCollection) + 0x305
backend-1-1  |    at Program.<>c__DisplayClass0_1.<<Main>$>b__20() + 0x22
backend-1-1  |    at Polly.Retry.RetryEngine.Implementation[TResult](Func`3, Context, CancellationToken, ExceptionPredicates, ResultPredicates`1, Action`4, Int32, IEnumerable`1, Func`4) + 0x92
backend-1-1  | ----------------------------------------
backend-2-1  | WORKER_SIZE: 15
backend-2-1  | BATCH_SIZE: 100
backend-2-1  | [Redis] Attempting connection...
backend-2-1  | Retry 1: RpcException - Status(StatusCode="Unavailable", Detail="Error connecting to subchannel.", DebugException="System.Net.Sockets.SocketException: Connection refused")
backend-2-1  | StackTrace:
backend-2-1  |    at Grpc.Net.Client.Balancer.Internal.ConnectionManager.<PickAsync>d__46.MoveNext() + 0x43f
backend-2-1  | --- End of stack trace from previous location ---
backend-2-1  |    at Grpc.Net.Client.Balancer.Internal.BalancerHttpHandler.<SendAsync>d__11.MoveNext() + 0x371
backend-2-1  | --- End of stack trace from previous location ---
backend-2-1  |    at Grpc.Net.Client.Internal.GrpcCall`2.<RunCall>d__82.MoveNext() + 0xefd
backend-2-1  | --- End of stack trace from previous location ---
backend-2-1  |    at MichelOliveira.Com.ReactiveLock.Distributed.Grpc.ReactiveLockGrpcTrackerStore.<SetStatusAsync>d__4.MoveNext() + 0x7a
backend-2-1  | --- End of stack trace from previous location ---
backend-2-1  |    at MichelOliveira.Com.ReactiveLock.Core.ReactiveLockTrackerController.<DecrementAsync>d__10.MoveNext() + 0x68
backend-2-1  | --- End of stack trace from previous location ---
backend-2-1  |    at MichelOliveira.Com.ReactiveLock.Distributed.Grpc.ReactiveLockGrpcTrackerExtensions.<UseDistributedGrpcReactiveLockAsync>d__10.MoveNext() + 0x2ac
backend-2-1  | --- End of stack trace from previous location ---
backend-2-1  |    at Program.<>c__DisplayClass0_0.<<<Main>$>b__21>d.MoveNext() + 0x50
backend-2-1  | --- End of stack trace from previous location ---
backend-2-1  |    at Polly.AsyncPolicy.<>c__DisplayClass40_0.<<ImplementationAsync>b__0>d.MoveNext() + 0x68
backend-2-1  | --- End of stack trace from previous location ---
backend-2-1  |    at Polly.Retry.AsyncRetryEngine.<ImplementationAsync>d__0`1.MoveNext() + 0x1c5
backend-2-1  | ----------------------------------------
backend-2-1  | Retry 1: RedisConnectionException - It was not possible to connect to the redis server(s). Error connecting right now. To allow this multiplexer to continue retrying until it's able to connect, use abortConnect=false in your connection string or AbortOnConnectFail=false; in your code.
backend-2-1  | StackTrace:
backend-2-1  |    at StackExchange.Redis.ConnectionMultiplexer.ConnectImpl(ConfigurationOptions, TextWriter, Nullable`1, EndPointCollection) + 0x305
backend-2-1  |    at Program.<>c__DisplayClass0_1.<<Main>$>b__20() + 0x22
backend-2-1  |    at Polly.Retry.RetryEngine.Implementation[TResult](Func`3, Context, CancellationToken, ExceptionPredicates, ResultPredicates`1, Action`4, Int32, IEnumerable`1, Func`4) + 0x92
backend-2-1  | ----------------------------------------
redis-1      | Starting Redis Server
redis-1      | 1:C 15 Aug 2025 16:03:43.833 # WARNING Memory overcommit must be enabled! Without it, a background save or replication may fail under low memory condition. Being disabled, it can also cause failures without low memory condition, see https://github.com/jemalloc/jemalloc/issues/1328. To fix this issue add 'vm.overcommit_memory = 1' to /etc/sysctl.conf and then reboot or run the command 'sysctl vm.overcommit_memory=1' for this to take effect.
redis-1      | 1:C 15 Aug 2025 16:03:43.834 * oO0OoO0OoO0Oo Redis is starting oO0OoO0OoO0Oo
redis-1      | 1:C 15 Aug 2025 16:03:43.834 * Redis version=8.2.0, bits=64, commit=00000000, modified=1, pid=1, just started
redis-1      | 1:C 15 Aug 2025 16:03:43.834 * Configuration loaded
redis-1      | 1:M 15 Aug 2025 16:03:43.835 * monotonic clock: POSIX clock_gettime
redis-1      | 1:M 15 Aug 2025 16:03:43.838 * Running mode=standalone, port=6379.
redis-1      | 1:M 15 Aug 2025 16:03:43.840 * <bf> RedisBloom version 8.2.0 (Git=unknown)
redis-1      | 1:M 15 Aug 2025 16:03:43.840 * <bf> Registering configuration options: [
redis-1      | 1:M 15 Aug 2025 16:03:43.840 * <bf> 	{ bf-error-rate       :      0.01 }
redis-1      | 1:M 15 Aug 2025 16:03:43.840 * <bf> 	{ bf-initial-size     :       100 }
redis-1      | 1:M 15 Aug 2025 16:03:43.840 * <bf> 	{ bf-expansion-factor :         2 }
redis-1      | 1:M 15 Aug 2025 16:03:43.840 * <bf> 	{ cf-bucket-size      :         2 }
redis-1      | 1:M 15 Aug 2025 16:03:43.840 * <bf> 	{ cf-initial-size     :      1024 }
redis-1      | 1:M 15 Aug 2025 16:03:43.840 * <bf> 	{ cf-max-iterations   :        20 }
redis-1      | 1:M 15 Aug 2025 16:03:43.840 * <bf> 	{ cf-expansion-factor :         1 }
redis-1      | 1:M 15 Aug 2025 16:03:43.840 * <bf> 	{ cf-max-expansions   :        32 }
redis-1      | 1:M 15 Aug 2025 16:03:43.840 * <bf> ]
redis-1      | 1:M 15 Aug 2025 16:03:43.840 * Module 'bf' loaded from /usr/local/lib/redis/modules//redisbloom.so
redis-1      | 1:M 15 Aug 2025 16:03:43.923 * <search> Redis version found by RedisSearch : 8.2.0 - oss
redis-1      | 1:M 15 Aug 2025 16:03:43.923 * <search> RediSearch version 8.2.0 (Git=d81ebce)
redis-1      | 1:M 15 Aug 2025 16:03:43.923 * <search> Low level api version 1 initialized successfully
redis-1      | 1:M 15 Aug 2025 16:03:43.923 * <search> gc: ON, prefix min length: 2, min word length to stem: 4, prefix max expansions: 200, query timeout (ms): 500, timeout policy: return, cursor read size: 1000, cursor max idle (ms): 300000, max doctable size: 1000000, max number of search results:  1000000, 
redis-1      | 1:M 15 Aug 2025 16:03:43.923 * <search> Initialized thread pools!
redis-1      | 1:M 15 Aug 2025 16:03:43.923 * <search> Disabled workers threadpool of size 0
redis-1      | 1:M 15 Aug 2025 16:03:43.924 * <search> Subscribe to config changes
redis-1      | 1:M 15 Aug 2025 16:03:43.924 * <search> Enabled role change notification
redis-1      | 1:M 15 Aug 2025 16:03:43.924 * <search> Cluster configuration: AUTO partitions, type: 0, coordinator timeout: 0ms
redis-1      | 1:M 15 Aug 2025 16:03:43.924 * <search> Register write commands
redis-1      | 1:M 15 Aug 2025 16:03:43.924 * Module 'search' loaded from /usr/local/lib/redis/modules//redisearch.so
redis-1      | 1:M 15 Aug 2025 16:03:43.927 * <timeseries> RedisTimeSeries version 80200, git_sha=1439d4a439ca9c063e6ef124a510abff09a5d493
redis-1      | 1:M 15 Aug 2025 16:03:43.927 * <timeseries> Redis version found by RedisTimeSeries : 8.2.0 - oss
redis-1      | 1:M 15 Aug 2025 16:03:43.927 * <timeseries> Registering configuration options: [
redis-1      | 1:M 15 Aug 2025 16:03:43.927 * <timeseries> 	{ ts-compaction-policy   :              }
redis-1      | 1:M 15 Aug 2025 16:03:43.927 * <timeseries> 	{ ts-num-threads         :            3 }
redis-1      | 1:M 15 Aug 2025 16:03:43.927 * <timeseries> 	{ ts-retention-policy    :            0 }
redis-1      | 1:M 15 Aug 2025 16:03:43.927 * <timeseries> 	{ ts-duplicate-policy    :        block }
redis-1      | 1:M 15 Aug 2025 16:03:43.927 * <timeseries> 	{ ts-chunk-size-bytes    :         4096 }
redis-1      | 1:M 15 Aug 2025 16:03:43.927 * <timeseries> 	{ ts-encoding            :   compressed }
redis-1      | 1:M 15 Aug 2025 16:03:43.927 * <timeseries> 	{ ts-ignore-max-time-diff:            0 }
redis-1      | 1:M 15 Aug 2025 16:03:43.927 * <timeseries> 	{ ts-ignore-max-val-diff :     0.000000 }
redis-1      | 1:M 15 Aug 2025 16:03:43.927 * <timeseries> ]
redis-1      | 1:M 15 Aug 2025 16:03:43.927 * <timeseries> Detected redis oss
redis-1      | 1:M 15 Aug 2025 16:03:43.927 * <timeseries> Enabled diskless replication
redis-1      | 1:M 15 Aug 2025 16:03:43.927 * Module 'timeseries' loaded from /usr/local/lib/redis/modules//redistimeseries.so
redis-1      | 1:M 15 Aug 2025 16:03:43.932 * <ReJSON> Created new data type 'ReJSON-RL'
redis-1      | 1:M 15 Aug 2025 16:03:43.933 * <ReJSON> version: 80200 git sha: unknown branch: unknown
redis-1      | 1:M 15 Aug 2025 16:03:43.933 * <ReJSON> Exported RedisJSON_V1 API
redis-1      | 1:M 15 Aug 2025 16:03:43.933 * <ReJSON> Exported RedisJSON_V2 API
redis-1      | 1:M 15 Aug 2025 16:03:43.933 * <ReJSON> Exported RedisJSON_V3 API
redis-1      | 1:M 15 Aug 2025 16:03:43.933 * <ReJSON> Exported RedisJSON_V4 API
redis-1      | 1:M 15 Aug 2025 16:03:43.933 * <ReJSON> Exported RedisJSON_V5 API
redis-1      | 1:M 15 Aug 2025 16:03:43.933 * <ReJSON> Enabled diskless replication
redis-1      | 1:M 15 Aug 2025 16:03:43.933 * <ReJSON> Initialized shared string cache, thread safe: false.
redis-1      | 1:M 15 Aug 2025 16:03:43.933 * Module 'ReJSON' loaded from /usr/local/lib/redis/modules//rejson.so
redis-1      | 1:M 15 Aug 2025 16:03:43.933 * <search> Acquired RedisJSON_V5 API
redis-1      | 1:M 15 Aug 2025 16:03:43.934 * Server initialized
redis-1      | 1:M 15 Aug 2025 16:03:43.934 * Ready to accept connections tcp
backend-1-1  | Retry 2: RpcException - Status(StatusCode="Unavailable", Detail="Error connecting to subchannel.", DebugException="System.Net.Sockets.SocketException: Connection refused")
backend-1-1  | StackTrace:
backend-1-1  |    at Grpc.Net.Client.Balancer.Internal.ConnectionManager.<PickAsync>d__46.MoveNext() + 0x43f
backend-1-1  | --- End of stack trace from previous location ---
backend-1-1  |    at Grpc.Net.Client.Balancer.Internal.BalancerHttpHandler.<SendAsync>d__11.MoveNext() + 0x371
backend-1-1  | --- End of stack trace from previous location ---
backend-1-1  |    at Grpc.Net.Client.Internal.GrpcCall`2.<RunCall>d__82.MoveNext() + 0xefd
backend-1-1  | --- End of stack trace from previous location ---
backend-1-1  |    at MichelOliveira.Com.ReactiveLock.Distributed.Grpc.ReactiveLockGrpcTrackerStore.<SetStatusAsync>d__4.MoveNext() + 0x7a
backend-1-1  | --- End of stack trace from previous location ---
backend-1-1  |    at MichelOliveira.Com.ReactiveLock.Core.ReactiveLockTrackerController.<DecrementAsync>d__10.MoveNext() + 0x68
backend-1-1  | --- End of stack trace from previous location ---
backend-1-1  |    at MichelOliveira.Com.ReactiveLock.Distributed.Grpc.ReactiveLockGrpcTrackerExtensions.<UseDistributedGrpcReactiveLockAsync>d__10.MoveNext() + 0x2ac
backend-1-1  | --- End of stack trace from previous location ---
backend-1-1  |    at Program.<>c__DisplayClass0_0.<<<Main>$>b__21>d.MoveNext() + 0x50
backend-1-1  | --- End of stack trace from previous location ---
backend-1-1  |    at Polly.AsyncPolicy.<>c__DisplayClass40_0.<<ImplementationAsync>b__0>d.MoveNext() + 0x68
backend-1-1  | --- End of stack trace from previous location ---
backend-1-1  |    at Polly.Retry.AsyncRetryEngine.<ImplementationAsync>d__0`1.MoveNext() + 0x1c5
backend-1-1  | ----------------------------------------
backend-1-1  | [Redis] Attempting connection...
backend-1-1  | [Redis] Connected successfully.
backend-2-1  | [Redis] Attempting connection...
backend-2-1  | Retry 2: RpcException - Status(StatusCode="Unavailable", Detail="Error connecting to subchannel.", DebugException="System.Net.Sockets.SocketException: Connection refused")
backend-2-1  | StackTrace:
backend-2-1  |    at Grpc.Net.Client.Balancer.Internal.ConnectionManager.<PickAsync>d__46.MoveNext() + 0x43f
backend-2-1  | --- End of stack trace from previous location ---
backend-2-1  |    at Grpc.Net.Client.Balancer.Internal.BalancerHttpHandler.<SendAsync>d__11.MoveNext() + 0x371
backend-2-1  | --- End of stack trace from previous location ---
backend-2-1  |    at Grpc.Net.Client.Internal.GrpcCall`2.<RunCall>d__82.MoveNext() + 0xefd
backend-2-1  | --- End of stack trace from previous location ---
backend-2-1  |    at MichelOliveira.Com.ReactiveLock.Distributed.Grpc.ReactiveLockGrpcTrackerStore.<SetStatusAsync>d__4.MoveNext() + 0x7a
backend-2-1  | --- End of stack trace from previous location ---
backend-2-1  |    at MichelOliveira.Com.ReactiveLock.Core.ReactiveLockTrackerController.<DecrementAsync>d__10.MoveNext() + 0x68
backend-2-1  | --- End of stack trace from previous location ---
backend-2-1  |    at MichelOliveira.Com.ReactiveLock.Distributed.Grpc.ReactiveLockGrpcTrackerExtensions.<UseDistributedGrpcReactiveLockAsync>d__10.MoveNext() + 0x2ac
backend-2-1  | --- End of stack trace from previous location ---
backend-2-1  |    at Program.<>c__DisplayClass0_0.<<<Main>$>b__21>d.MoveNext() + 0x50
backend-2-1  | --- End of stack trace from previous location ---
backend-2-1  |    at Polly.AsyncPolicy.<>c__DisplayClass40_0.<<ImplementationAsync>b__0>d.MoveNext() + 0x68
backend-2-1  | --- End of stack trace from previous location ---
backend-2-1  |    at Polly.Retry.AsyncRetryEngine.<ImplementationAsync>d__0`1.MoveNext() + 0x1c5
backend-2-1  | ----------------------------------------
backend-2-1  | [Redis] Connected successfully.
backend-1-1  | Retry 3: RpcException - Status(StatusCode="Unavailable", Detail="Error connecting to subchannel.", DebugException="System.Net.Sockets.SocketException: Connection refused")
backend-1-1  | StackTrace:
backend-1-1  |    at Grpc.Net.Client.Balancer.Internal.ConnectionManager.<PickAsync>d__46.MoveNext() + 0x43f
backend-1-1  | --- End of stack trace from previous location ---
backend-1-1  |    at Grpc.Net.Client.Balancer.Internal.BalancerHttpHandler.<SendAsync>d__11.MoveNext() + 0x371
backend-1-1  | --- End of stack trace from previous location ---
backend-1-1  |    at Grpc.Net.Client.Internal.GrpcCall`2.<RunCall>d__82.MoveNext() + 0xefd
backend-1-1  | --- End of stack trace from previous location ---
backend-1-1  |    at MichelOliveira.Com.ReactiveLock.Distributed.Grpc.ReactiveLockGrpcTrackerStore.<SetStatusAsync>d__4.MoveNext() + 0x7a
backend-1-1  | --- End of stack trace from previous location ---
backend-1-1  |    at MichelOliveira.Com.ReactiveLock.Core.ReactiveLockTrackerController.<DecrementAsync>d__10.MoveNext() + 0x68
backend-1-1  | --- End of stack trace from previous location ---
backend-1-1  |    at MichelOliveira.Com.ReactiveLock.Distributed.Grpc.ReactiveLockGrpcTrackerExtensions.<UseDistributedGrpcReactiveLockAsync>d__10.MoveNext() + 0x2ac
backend-1-1  | --- End of stack trace from previous location ---
backend-1-1  |    at Program.<>c__DisplayClass0_0.<<<Main>$>b__21>d.MoveNext() + 0x50
backend-1-1  | --- End of stack trace from previous location ---
backend-1-1  |    at Polly.AsyncPolicy.<>c__DisplayClass40_0.<<ImplementationAsync>b__0>d.MoveNext() + 0x68
backend-1-1  | --- End of stack trace from previous location ---
backend-1-1  |    at Polly.Retry.AsyncRetryEngine.<ImplementationAsync>d__0`1.MoveNext() + 0x1c5
backend-1-1  | ----------------------------------------
load-balancer-1  | /docker-entrypoint.sh: /docker-entrypoint.d/ is not empty, will attempt to perform configuration
load-balancer-1  | /docker-entrypoint.sh: Looking for shell scripts in /docker-entrypoint.d/
load-balancer-1  | /docker-entrypoint.sh: Launching /docker-entrypoint.d/10-listen-on-ipv6-by-default.sh
load-balancer-1  | 10-listen-on-ipv6-by-default.sh: info: Getting the checksum of /etc/nginx/conf.d/default.conf
load-balancer-1  | 10-listen-on-ipv6-by-default.sh: info: Enabled listen on IPv6 in /etc/nginx/conf.d/default.conf
load-balancer-1  | /docker-entrypoint.sh: Sourcing /docker-entrypoint.d/15-local-resolvers.envsh
load-balancer-1  | /docker-entrypoint.sh: Launching /docker-entrypoint.d/20-envsubst-on-templates.sh
load-balancer-1  | /docker-entrypoint.sh: Launching /docker-entrypoint.d/30-tune-worker-processes.sh
load-balancer-1  | /docker-entrypoint.sh: Configuration complete; ready for start up
redis-1          | 1:signal-handler (1755273898) Received SIGTERM scheduling shutdown...
redis-1          | 1:M 15 Aug 2025 16:04:58.681 * User requested shutdown...
redis-1          | 1:M 15 Aug 2025 16:04:58.682 # Redis is now ready to exit, bye bye...
backend-1-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 75) on redis:6379/Interactive, Idle/Faulted, last: LPOP, origin: ReadFromPipe, outstanding: 15, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 8 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-1-1      |  ---> Pipelines.Sockets.Unofficial.ConnectionResetException: Connection reset by peer
backend-1-1      |  ---> System.Net.Sockets.SocketException (104): Connection reset by peer
backend-1-1      |    at Pipelines.Sockets.Unofficial.Internal.Throw.Socket(Int32) + 0x27
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketAwaitableEventArgs.GetResult() + 0x3e
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketConnection.<DoReceiveAsync>d__73.MoveNext() + 0xec
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadResult(ReadResult&) + 0x170
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadAsyncResult() + 0x96
backend-1-1      |    at StackExchange.Redis.PhysicalConnection.<ReadFromPipe>d__134.MoveNext() + 0x2e2
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x154
backend-1-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 75) on redis:6379/Interactive, Idle/Faulted, last: LPOP, origin: ReadFromPipe, outstanding: 15, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 8 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-1-1      |  ---> Pipelines.Sockets.Unofficial.ConnectionResetException: Connection reset by peer
backend-1-1      |  ---> System.Net.Sockets.SocketException (104): Connection reset by peer
backend-1-1      |    at Pipelines.Sockets.Unofficial.Internal.Throw.Socket(Int32) + 0x27
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketAwaitableEventArgs.GetResult() + 0x3e
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketConnection.<DoReceiveAsync>d__73.MoveNext() + 0xec
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadResult(ReadResult&) + 0x170
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadAsyncResult() + 0x96
backend-1-1      |    at StackExchange.Redis.PhysicalConnection.<ReadFromPipe>d__134.MoveNext() + 0x2e2
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x154
backend-1-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 75) on redis:6379/Interactive, Idle/Faulted, last: LPOP, origin: ReadFromPipe, outstanding: 15, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 8 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-1-1      |  ---> Pipelines.Sockets.Unofficial.ConnectionResetException: Connection reset by peer
backend-1-1      |  ---> System.Net.Sockets.SocketException (104): Connection reset by peer
backend-1-1      |    at Pipelines.Sockets.Unofficial.Internal.Throw.Socket(Int32) + 0x27
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketAwaitableEventArgs.GetResult() + 0x3e
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketConnection.<DoReceiveAsync>d__73.MoveNext() + 0xec
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadResult(ReadResult&) + 0x170
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadAsyncResult() + 0x96
backend-1-1      |    at StackExchange.Redis.PhysicalConnection.<ReadFromPipe>d__134.MoveNext() + 0x2e2
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x154
backend-1-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 75) on redis:6379/Interactive, Idle/Faulted, last: LPOP, origin: ReadFromPipe, outstanding: 15, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 8 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-1-1      |  ---> Pipelines.Sockets.Unofficial.ConnectionResetException: Connection reset by peer
backend-1-1      |  ---> System.Net.Sockets.SocketException (104): Connection reset by peer
backend-1-1      |    at Pipelines.Sockets.Unofficial.Internal.Throw.Socket(Int32) + 0x27
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketAwaitableEventArgs.GetResult() + 0x3e
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketConnection.<DoReceiveAsync>d__73.MoveNext() + 0xec
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadResult(ReadResult&) + 0x170
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadAsyncResult() + 0x96
backend-1-1      |    at StackExchange.Redis.PhysicalConnection.<ReadFromPipe>d__134.MoveNext() + 0x2e2
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x154
backend-1-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 75) on redis:6379/Interactive, Idle/Faulted, last: LPOP, origin: ReadFromPipe, outstanding: 15, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 8 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-1-1      |  ---> Pipelines.Sockets.Unofficial.ConnectionResetException: Connection reset by peer
backend-1-1      |  ---> System.Net.Sockets.SocketException (104): Connection reset by peer
backend-1-1      |    at Pipelines.Sockets.Unofficial.Internal.Throw.Socket(Int32) + 0x27
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketAwaitableEventArgs.GetResult() + 0x3e
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketConnection.<DoReceiveAsync>d__73.MoveNext() + 0xec
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadResult(ReadResult&) + 0x170
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadAsyncResult() + 0x96
backend-1-1      |    at StackExchange.Redis.PhysicalConnection.<ReadFromPipe>d__134.MoveNext() + 0x2e2
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x154
backend-1-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 75) on redis:6379/Interactive, Idle/Faulted, last: LPOP, origin: ReadFromPipe, outstanding: 15, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 8 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-1-1      |  ---> Pipelines.Sockets.Unofficial.ConnectionResetException: Connection reset by peer
backend-1-1      |  ---> System.Net.Sockets.SocketException (104): Connection reset by peer
backend-1-1      |    at Pipelines.Sockets.Unofficial.Internal.Throw.Socket(Int32) + 0x27
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketAwaitableEventArgs.GetResult() + 0x3e
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketConnection.<DoReceiveAsync>d__73.MoveNext() + 0xec
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadResult(ReadResult&) + 0x170
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadAsyncResult() + 0x96
backend-1-1      |    at StackExchange.Redis.PhysicalConnection.<ReadFromPipe>d__134.MoveNext() + 0x2e2
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x154
backend-1-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 75) on redis:6379/Interactive, Idle/Faulted, last: LPOP, origin: ReadFromPipe, outstanding: 15, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 8 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-1-1      |  ---> Pipelines.Sockets.Unofficial.ConnectionResetException: Connection reset by peer
backend-1-1      |  ---> System.Net.Sockets.SocketException (104): Connection reset by peer
backend-1-1      |    at Pipelines.Sockets.Unofficial.Internal.Throw.Socket(Int32) + 0x27
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketAwaitableEventArgs.GetResult() + 0x3e
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketConnection.<DoReceiveAsync>d__73.MoveNext() + 0xec
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadResult(ReadResult&) + 0x170
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadAsyncResult() + 0x96
backend-1-1      |    at StackExchange.Redis.PhysicalConnection.<ReadFromPipe>d__134.MoveNext() + 0x2e2
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x154
backend-1-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 75) on redis:6379/Interactive, Idle/Faulted, last: LPOP, origin: ReadFromPipe, outstanding: 15, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 8 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-1-1      |  ---> Pipelines.Sockets.Unofficial.ConnectionResetException: Connection reset by peer
backend-1-1      |  ---> System.Net.Sockets.SocketException (104): Connection reset by peer
backend-1-1      |    at Pipelines.Sockets.Unofficial.Internal.Throw.Socket(Int32) + 0x27
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketAwaitableEventArgs.GetResult() + 0x3e
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketConnection.<DoReceiveAsync>d__73.MoveNext() + 0xec
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadResult(ReadResult&) + 0x170
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadAsyncResult() + 0x96
backend-1-1      |    at StackExchange.Redis.PhysicalConnection.<ReadFromPipe>d__134.MoveNext() + 0x2e2
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x154
backend-1-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 75) on redis:6379/Interactive, Idle/Faulted, last: LPOP, origin: ReadFromPipe, outstanding: 15, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 8 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-1-1      |  ---> Pipelines.Sockets.Unofficial.ConnectionResetException: Connection reset by peer
backend-1-1      |  ---> System.Net.Sockets.SocketException (104): Connection reset by peer
backend-1-1      |    at Pipelines.Sockets.Unofficial.Internal.Throw.Socket(Int32) + 0x27
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketAwaitableEventArgs.GetResult() + 0x3e
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketConnection.<DoReceiveAsync>d__73.MoveNext() + 0xec
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadResult(ReadResult&) + 0x170
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadAsyncResult() + 0x96
backend-1-1      |    at StackExchange.Redis.PhysicalConnection.<ReadFromPipe>d__134.MoveNext() + 0x2e2
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x154
backend-1-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 75) on redis:6379/Interactive, Idle/Faulted, last: LPOP, origin: ReadFromPipe, outstanding: 15, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 8 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-1-1      |  ---> Pipelines.Sockets.Unofficial.ConnectionResetException: Connection reset by peer
backend-1-1      |  ---> System.Net.Sockets.SocketException (104): Connection reset by peer
backend-1-1      |    at Pipelines.Sockets.Unofficial.Internal.Throw.Socket(Int32) + 0x27
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketAwaitableEventArgs.GetResult() + 0x3e
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketConnection.<DoReceiveAsync>d__73.MoveNext() + 0xec
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadResult(ReadResult&) + 0x170
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadAsyncResult() + 0x96
backend-1-1      |    at StackExchange.Redis.PhysicalConnection.<ReadFromPipe>d__134.MoveNext() + 0x2e2
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x154
backend-1-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 75) on redis:6379/Interactive, Idle/Faulted, last: LPOP, origin: ReadFromPipe, outstanding: 15, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 8 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-1-1      |  ---> Pipelines.Sockets.Unofficial.ConnectionResetException: Connection reset by peer
backend-1-1      |  ---> System.Net.Sockets.SocketException (104): Connection reset by peer
backend-1-1      |    at Pipelines.Sockets.Unofficial.Internal.Throw.Socket(Int32) + 0x27
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketAwaitableEventArgs.GetResult() + 0x3e
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketConnection.<DoReceiveAsync>d__73.MoveNext() + 0xec
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadResult(ReadResult&) + 0x170
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadAsyncResult() + 0x96
backend-1-1      |    at StackExchange.Redis.PhysicalConnection.<ReadFromPipe>d__134.MoveNext() + 0x2e2
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x154
backend-1-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 75) on redis:6379/Interactive, Idle/Faulted, last: LPOP, origin: ReadFromPipe, outstanding: 15, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 8 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-1-1      |  ---> Pipelines.Sockets.Unofficial.ConnectionResetException: Connection reset by peer
backend-1-1      |  ---> System.Net.Sockets.SocketException (104): Connection reset by peer
backend-1-1      |    at Pipelines.Sockets.Unofficial.Internal.Throw.Socket(Int32) + 0x27
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketAwaitableEventArgs.GetResult() + 0x3e
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketConnection.<DoReceiveAsync>d__73.MoveNext() + 0xec
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadResult(ReadResult&) + 0x170
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadAsyncResult() + 0x96
backend-1-1      |    at StackExchange.Redis.PhysicalConnection.<ReadFromPipe>d__134.MoveNext() + 0x2e2
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x154
backend-1-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 75) on redis:6379/Interactive, Idle/Faulted, last: LPOP, origin: ReadFromPipe, outstanding: 15, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 8 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-1-1      |  ---> Pipelines.Sockets.Unofficial.ConnectionResetException: Connection reset by peer
backend-1-1      |  ---> System.Net.Sockets.SocketException (104): Connection reset by peer
backend-1-1      |    at Pipelines.Sockets.Unofficial.Internal.Throw.Socket(Int32) + 0x27
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketAwaitableEventArgs.GetResult() + 0x3e
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketConnection.<DoReceiveAsync>d__73.MoveNext() + 0xec
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadResult(ReadResult&) + 0x170
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadAsyncResult() + 0x96
backend-1-1      |    at StackExchange.Redis.PhysicalConnection.<ReadFromPipe>d__134.MoveNext() + 0x2e2
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x154
backend-2-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 75) on redis:6379/Interactive, Idle/Faulted, last: LPOP, origin: ReadFromPipe, outstanding: 15, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 9 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-2-1      |  ---> Pipelines.Sockets.Unofficial.ConnectionResetException: Connection reset by peer
backend-2-1      |  ---> System.Net.Sockets.SocketException (104): Connection reset by peer
backend-2-1      |    at Pipelines.Sockets.Unofficial.Internal.Throw.Socket(Int32) + 0x27
backend-2-1      |    at Pipelines.Sockets.Unofficial.SocketAwaitableEventArgs.GetResult() + 0x3e
backend-2-1      |    at Pipelines.Sockets.Unofficial.SocketConnection.<DoReceiveAsync>d__73.MoveNext() + 0xec
backend-2-1      |    --- End of inner exception stack trace ---
backend-2-1      |    at System.IO.Pipelines.Pipe.GetReadResult(ReadResult&) + 0x170
backend-2-1      |    at System.IO.Pipelines.Pipe.GetReadAsyncResult() + 0x96
backend-2-1      |    at StackExchange.Redis.PhysicalConnection.<ReadFromPipe>d__134.MoveNext() + 0x2e2
backend-2-1      |    --- End of inner exception stack trace ---
backend-2-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x154
backend-2-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 75) on redis:6379/Interactive, Idle/Faulted, last: LPOP, origin: ReadFromPipe, outstanding: 15, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 9 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-2-1      |  ---> Pipelines.Sockets.Unofficial.ConnectionResetException: Connection reset by peer
backend-2-1      |  ---> System.Net.Sockets.SocketException (104): Connection reset by peer
backend-2-1      |    at Pipelines.Sockets.Unofficial.Internal.Throw.Socket(Int32) + 0x27
backend-2-1      |    at Pipelines.Sockets.Unofficial.SocketAwaitableEventArgs.GetResult() + 0x3e
backend-2-1      |    at Pipelines.Sockets.Unofficial.SocketConnection.<DoReceiveAsync>d__73.MoveNext() + 0xec
backend-2-1      |    --- End of inner exception stack trace ---
backend-2-1      |    at System.IO.Pipelines.Pipe.GetReadResult(ReadResult&) + 0x170
backend-2-1      |    at System.IO.Pipelines.Pipe.GetReadAsyncResult() + 0x96
backend-2-1      |    at StackExchange.Redis.PhysicalConnection.<ReadFromPipe>d__134.MoveNext() + 0x2e2
backend-2-1      |    --- End of inner exception stack trace ---
backend-2-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x154
backend-2-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 75) on redis:6379/Interactive, Idle/Faulted, last: LPOP, origin: ReadFromPipe, outstanding: 15, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 9 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-2-1      |  ---> Pipelines.Sockets.Unofficial.ConnectionResetException: Connection reset by peer
backend-2-1      |  ---> System.Net.Sockets.SocketException (104): Connection reset by peer
backend-2-1      |    at Pipelines.Sockets.Unofficial.Internal.Throw.Socket(Int32) + 0x27
backend-2-1      |    at Pipelines.Sockets.Unofficial.SocketAwaitableEventArgs.GetResult() + 0x3e
backend-2-1      |    at Pipelines.Sockets.Unofficial.SocketConnection.<DoReceiveAsync>d__73.MoveNext() + 0xec
backend-2-1      |    --- End of inner exception stack trace ---
backend-2-1      |    at System.IO.Pipelines.Pipe.GetReadResult(ReadResult&) + 0x170
backend-2-1      |    at System.IO.Pipelines.Pipe.GetReadAsyncResult() + 0x96
backend-2-1      |    at StackExchange.Redis.PhysicalConnection.<ReadFromPipe>d__134.MoveNext() + 0x2e2
backend-2-1      |    --- End of inner exception stack trace ---
backend-2-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x154
backend-2-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 75) on redis:6379/Interactive, Idle/Faulted, last: LPOP, origin: ReadFromPipe, outstanding: 15, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 9 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-2-1      |  ---> Pipelines.Sockets.Unofficial.ConnectionResetException: Connection reset by peer
backend-2-1      |  ---> System.Net.Sockets.SocketException (104): Connection reset by peer
backend-2-1      |    at Pipelines.Sockets.Unofficial.Internal.Throw.Socket(Int32) + 0x27
backend-2-1      |    at Pipelines.Sockets.Unofficial.SocketAwaitableEventArgs.GetResult() + 0x3e
backend-2-1      |    at Pipelines.Sockets.Unofficial.SocketConnection.<DoReceiveAsync>d__73.MoveNext() + 0xec
backend-2-1      |    --- End of inner exception stack trace ---
backend-2-1      |    at System.IO.Pipelines.Pipe.GetReadResult(ReadResult&) + 0x170
backend-2-1      |    at System.IO.Pipelines.Pipe.GetReadAsyncResult() + 0x96
backend-2-1      |    at StackExchange.Redis.PhysicalConnection.<ReadFromPipe>d__134.MoveNext() + 0x2e2
backend-2-1      |    --- End of inner exception stack trace ---
backend-2-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x154
backend-2-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 75) on redis:6379/Interactive, Idle/Faulted, last: LPOP, origin: ReadFromPipe, outstanding: 15, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 9 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-2-1      |  ---> Pipelines.Sockets.Unofficial.ConnectionResetException: Connection reset by peer
backend-2-1      |  ---> System.Net.Sockets.SocketException (104): Connection reset by peer
backend-2-1      |    at Pipelines.Sockets.Unofficial.Internal.Throw.Socket(Int32) + 0x27
backend-2-1      |    at Pipelines.Sockets.Unofficial.SocketAwaitableEventArgs.GetResult() + 0x3e
backend-2-1      |    at Pipelines.Sockets.Unofficial.SocketConnection.<DoReceiveAsync>d__73.MoveNext() + 0xec
backend-2-1      |    --- End of inner exception stack trace ---
backend-2-1      |    at System.IO.Pipelines.Pipe.GetReadResult(ReadResult&) + 0x170
backend-2-1      |    at System.IO.Pipelines.Pipe.GetReadAsyncResult() + 0x96
backend-2-1      |    at StackExchange.Redis.PhysicalConnection.<ReadFromPipe>d__134.MoveNext() + 0x2e2
backend-2-1      |    --- End of inner exception stack trace ---
backend-2-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x154
backend-2-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 75) on redis:6379/Interactive, Idle/Faulted, last: LPOP, origin: ReadFromPipe, outstanding: 15, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 9 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-2-1      |  ---> Pipelines.Sockets.Unofficial.ConnectionResetException: Connection reset by peer
backend-2-1      |  ---> System.Net.Sockets.SocketException (104): Connection reset by peer
backend-2-1      |    at Pipelines.Sockets.Unofficial.Internal.Throw.Socket(Int32) + 0x27
backend-2-1      |    at Pipelines.Sockets.Unofficial.SocketAwaitableEventArgs.GetResult() + 0x3e
backend-2-1      |    at Pipelines.Sockets.Unofficial.SocketConnection.<DoReceiveAsync>d__73.MoveNext() + 0xec
backend-2-1      |    --- End of inner exception stack trace ---
backend-2-1      |    at System.IO.Pipelines.Pipe.GetReadResult(ReadResult&) + 0x170
backend-2-1      |    at System.IO.Pipelines.Pipe.GetReadAsyncResult() + 0x96
backend-2-1      |    at StackExchange.Redis.PhysicalConnection.<ReadFromPipe>d__134.MoveNext() + 0x2e2
backend-2-1      |    --- End of inner exception stack trace ---
backend-2-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x154
backend-2-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 75) on redis:6379/Interactive, Idle/Faulted, last: LPOP, origin: ReadFromPipe, outstanding: 15, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 9 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-2-1      |  ---> Pipelines.Sockets.Unofficial.ConnectionResetException: Connection reset by peer
backend-2-1      |  ---> System.Net.Sockets.SocketException (104): Connection reset by peer
backend-2-1      |    at Pipelines.Sockets.Unofficial.Internal.Throw.Socket(Int32) + 0x27
backend-2-1      |    at Pipelines.Sockets.Unofficial.SocketAwaitableEventArgs.GetResult() + 0x3e
backend-2-1      |    at Pipelines.Sockets.Unofficial.SocketConnection.<DoReceiveAsync>d__73.MoveNext() + 0xec
backend-2-1      |    --- End of inner exception stack trace ---
backend-2-1      |    at System.IO.Pipelines.Pipe.GetReadResult(ReadResult&) + 0x170
backend-2-1      |    at System.IO.Pipelines.Pipe.GetReadAsyncResult() + 0x96
backend-2-1      |    at StackExchange.Redis.PhysicalConnection.<ReadFromPipe>d__134.MoveNext() + 0x2e2
backend-2-1      |    --- End of inner exception stack trace ---
backend-2-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x154
backend-2-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 75) on redis:6379/Interactive, Idle/Faulted, last: LPOP, origin: ReadFromPipe, outstanding: 15, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 9 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-2-1      |  ---> Pipelines.Sockets.Unofficial.ConnectionResetException: Connection reset by peer
backend-2-1      |  ---> System.Net.Sockets.SocketException (104): Connection reset by peer
backend-2-1      |    at Pipelines.Sockets.Unofficial.Internal.Throw.Socket(Int32) + 0x27
backend-2-1      |    at Pipelines.Sockets.Unofficial.SocketAwaitableEventArgs.GetResult() + 0x3e
backend-2-1      |    at Pipelines.Sockets.Unofficial.SocketConnection.<DoReceiveAsync>d__73.MoveNext() + 0xec
backend-2-1      |    --- End of inner exception stack trace ---
backend-2-1      |    at System.IO.Pipelines.Pipe.GetReadResult(ReadResult&) + 0x170
backend-2-1      |    at System.IO.Pipelines.Pipe.GetReadAsyncResult() + 0x96
backend-2-1      |    at StackExchange.Redis.PhysicalConnection.<ReadFromPipe>d__134.MoveNext() + 0x2e2
backend-2-1      |    --- End of inner exception stack trace ---
backend-2-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x154
backend-2-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 75) on redis:6379/Interactive, Idle/Faulted, last: LPOP, origin: ReadFromPipe, outstanding: 15, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 9 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-2-1      |  ---> Pipelines.Sockets.Unofficial.ConnectionResetException: Connection reset by peer
backend-2-1      |  ---> System.Net.Sockets.SocketException (104): Connection reset by peer
backend-2-1      |    at Pipelines.Sockets.Unofficial.Internal.Throw.Socket(Int32) + 0x27
backend-2-1      |    at Pipelines.Sockets.Unofficial.SocketAwaitableEventArgs.GetResult() + 0x3e
backend-2-1      |    at Pipelines.Sockets.Unofficial.SocketConnection.<DoReceiveAsync>d__73.MoveNext() + 0xec
backend-2-1      |    --- End of inner exception stack trace ---
backend-2-1      |    at System.IO.Pipelines.Pipe.GetReadResult(ReadResult&) + 0x170
backend-2-1      |    at System.IO.Pipelines.Pipe.GetReadAsyncResult() + 0x96
backend-2-1      |    at StackExchange.Redis.PhysicalConnection.<ReadFromPipe>d__134.MoveNext() + 0x2e2
backend-2-1      |    --- End of inner exception stack trace ---
backend-2-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x154
backend-2-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 75) on redis:6379/Interactive, Idle/Faulted, last: LPOP, origin: ReadFromPipe, outstanding: 15, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 9 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-2-1      |  ---> Pipelines.Sockets.Unofficial.ConnectionResetException: Connection reset by peer
backend-2-1      |  ---> System.Net.Sockets.SocketException (104): Connection reset by peer
backend-2-1      |    at Pipelines.Sockets.Unofficial.Internal.Throw.Socket(Int32) + 0x27
backend-2-1      |    at Pipelines.Sockets.Unofficial.SocketAwaitableEventArgs.GetResult() + 0x3e
backend-2-1      |    at Pipelines.Sockets.Unofficial.SocketConnection.<DoReceiveAsync>d__73.MoveNext() + 0xec
backend-2-1      |    --- End of inner exception stack trace ---
backend-2-1      |    at System.IO.Pipelines.Pipe.GetReadResult(ReadResult&) + 0x170
backend-2-1      |    at System.IO.Pipelines.Pipe.GetReadAsyncResult() + 0x96
backend-2-1      |    at StackExchange.Redis.PhysicalConnection.<ReadFromPipe>d__134.MoveNext() + 0x2e2
backend-2-1      |    --- End of inner exception stack trace ---
backend-2-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x154
backend-2-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 75) on redis:6379/Interactive, Idle/Faulted, last: LPOP, origin: ReadFromPipe, outstanding: 15, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 9 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-2-1      |  ---> Pipelines.Sockets.Unofficial.ConnectionResetException: Connection reset by peer
backend-2-1      |  ---> System.Net.Sockets.SocketException (104): Connection reset by peer
backend-2-1      |    at Pipelines.Sockets.Unofficial.Internal.Throw.Socket(Int32) + 0x27
backend-2-1      |    at Pipelines.Sockets.Unofficial.SocketAwaitableEventArgs.GetResult() + 0x3e
backend-2-1      |    at Pipelines.Sockets.Unofficial.SocketConnection.<DoReceiveAsync>d__73.MoveNext() + 0xec
backend-2-1      |    --- End of inner exception stack trace ---
backend-2-1      |    at System.IO.Pipelines.Pipe.GetReadResult(ReadResult&) + 0x170
backend-2-1      |    at System.IO.Pipelines.Pipe.GetReadAsyncResult() + 0x96
backend-2-1      |    at StackExchange.Redis.PhysicalConnection.<ReadFromPipe>d__134.MoveNext() + 0x2e2
backend-2-1      |    --- End of inner exception stack trace ---
backend-2-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x154
backend-2-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 75) on redis:6379/Interactive, Idle/Faulted, last: LPOP, origin: ReadFromPipe, outstanding: 15, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 9 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-2-1      |  ---> Pipelines.Sockets.Unofficial.ConnectionResetException: Connection reset by peer
backend-2-1      |  ---> System.Net.Sockets.SocketException (104): Connection reset by peer
backend-2-1      |    at Pipelines.Sockets.Unofficial.Internal.Throw.Socket(Int32) + 0x27
backend-2-1      |    at Pipelines.Sockets.Unofficial.SocketAwaitableEventArgs.GetResult() + 0x3e
backend-2-1      |    at Pipelines.Sockets.Unofficial.SocketConnection.<DoReceiveAsync>d__73.MoveNext() + 0xec
backend-2-1      |    --- End of inner exception stack trace ---
backend-2-1      |    at System.IO.Pipelines.Pipe.GetReadResult(ReadResult&) + 0x170
backend-2-1      |    at System.IO.Pipelines.Pipe.GetReadAsyncResult() + 0x96
backend-2-1      |    at StackExchange.Redis.PhysicalConnection.<ReadFromPipe>d__134.MoveNext() + 0x2e2
backend-2-1      |    --- End of inner exception stack trace ---
backend-2-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x154
backend-2-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 75) on redis:6379/Interactive, Idle/Faulted, last: LPOP, origin: ReadFromPipe, outstanding: 15, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 9 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-2-1      |  ---> Pipelines.Sockets.Unofficial.ConnectionResetException: Connection reset by peer
backend-2-1      |  ---> System.Net.Sockets.SocketException (104): Connection reset by peer
backend-2-1      |    at Pipelines.Sockets.Unofficial.Internal.Throw.Socket(Int32) + 0x27
backend-2-1      |    at Pipelines.Sockets.Unofficial.SocketAwaitableEventArgs.GetResult() + 0x3e
backend-2-1      |    at Pipelines.Sockets.Unofficial.SocketConnection.<DoReceiveAsync>d__73.MoveNext() + 0xec
backend-2-1      |    --- End of inner exception stack trace ---
backend-2-1      |    at System.IO.Pipelines.Pipe.GetReadResult(ReadResult&) + 0x170
backend-2-1      |    at System.IO.Pipelines.Pipe.GetReadAsyncResult() + 0x96
backend-2-1      |    at StackExchange.Redis.PhysicalConnection.<ReadFromPipe>d__134.MoveNext() + 0x2e2
backend-2-1      |    --- End of inner exception stack trace ---
backend-2-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x154
backend-2-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 75) on redis:6379/Interactive, Idle/Faulted, last: LPOP, origin: ReadFromPipe, outstanding: 15, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 9 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-2-1      |  ---> Pipelines.Sockets.Unofficial.ConnectionResetException: Connection reset by peer
backend-2-1      |  ---> System.Net.Sockets.SocketException (104): Connection reset by peer
backend-2-1      |    at Pipelines.Sockets.Unofficial.Internal.Throw.Socket(Int32) + 0x27
backend-2-1      |    at Pipelines.Sockets.Unofficial.SocketAwaitableEventArgs.GetResult() + 0x3e
backend-2-1      |    at Pipelines.Sockets.Unofficial.SocketConnection.<DoReceiveAsync>d__73.MoveNext() + 0xec
backend-2-1      |    --- End of inner exception stack trace ---
backend-2-1      |    at System.IO.Pipelines.Pipe.GetReadResult(ReadResult&) + 0x170
backend-2-1      |    at System.IO.Pipelines.Pipe.GetReadAsyncResult() + 0x96
backend-2-1      |    at StackExchange.Redis.PhysicalConnection.<ReadFromPipe>d__134.MoveNext() + 0x2e2
backend-2-1      |    --- End of inner exception stack trace ---
backend-2-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x154
backend-2-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 75) on redis:6379/Interactive, Idle/Faulted, last: LPOP, origin: ReadFromPipe, outstanding: 15, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 9 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-2-1      |  ---> Pipelines.Sockets.Unofficial.ConnectionResetException: Connection reset by peer
backend-2-1      |  ---> System.Net.Sockets.SocketException (104): Connection reset by peer
backend-2-1      |    at Pipelines.Sockets.Unofficial.Internal.Throw.Socket(Int32) + 0x27
backend-2-1      |    at Pipelines.Sockets.Unofficial.SocketAwaitableEventArgs.GetResult() + 0x3e
backend-2-1      |    at Pipelines.Sockets.Unofficial.SocketConnection.<DoReceiveAsync>d__73.MoveNext() + 0xec
backend-2-1      |    --- End of inner exception stack trace ---
backend-2-1      |    at System.IO.Pipelines.Pipe.GetReadResult(ReadResult&) + 0x170
backend-2-1      |    at System.IO.Pipelines.Pipe.GetReadAsyncResult() + 0x96
backend-2-1      |    at StackExchange.Redis.PhysicalConnection.<ReadFromPipe>d__134.MoveNext() + 0x2e2
backend-2-1      |    --- End of inner exception stack trace ---
backend-2-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x154
backend-1-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 75) on redis:6379/Interactive, Idle/Faulted, last: LPOP, origin: ReadFromPipe, outstanding: 15, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 8 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-1-1      |  ---> Pipelines.Sockets.Unofficial.ConnectionResetException: Connection reset by peer
backend-1-1      |  ---> System.Net.Sockets.SocketException (104): Connection reset by peer
backend-1-1      |    at Pipelines.Sockets.Unofficial.Internal.Throw.Socket(Int32) + 0x27
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketAwaitableEventArgs.GetResult() + 0x3e
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketConnection.<DoReceiveAsync>d__73.MoveNext() + 0xec
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadResult(ReadResult&) + 0x170
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadAsyncResult() + 0x96
backend-1-1      |    at StackExchange.Redis.PhysicalConnection.<ReadFromPipe>d__134.MoveNext() + 0x2e2
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x154
backend-1-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: SocketFailure (ReadSocketError/ConnectionReset, last-recv: 75) on redis:6379/Interactive, Idle/Faulted, last: LPOP, origin: ReadFromPipe, outstanding: 15, last-read: 0s ago, last-write: 0s ago, unanswered-write: 0s ago, keep-alive: 60s, state: ConnectedEstablished, mgr: 8 of 10 available, in: 0, in-pipe: 0, out-pipe: 0, last-heartbeat: 0s ago, last-mbeat: 0s ago, global: 0s ago, v: 2.8.58.19568
backend-1-1      |  ---> Pipelines.Sockets.Unofficial.ConnectionResetException: Connection reset by peer
backend-1-1      |  ---> System.Net.Sockets.SocketException (104): Connection reset by peer
backend-1-1      |    at Pipelines.Sockets.Unofficial.Internal.Throw.Socket(Int32) + 0x27
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketAwaitableEventArgs.GetResult() + 0x3e
backend-1-1      |    at Pipelines.Sockets.Unofficial.SocketConnection.<DoReceiveAsync>d__73.MoveNext() + 0xec
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadResult(ReadResult&) + 0x170
backend-1-1      |    at System.IO.Pipelines.Pipe.GetReadAsyncResult() + 0x96
backend-1-1      |    at StackExchange.Redis.PhysicalConnection.<ReadFromPipe>d__134.MoveNext() + 0x2e2
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x154
[Kload-balancer-1 exited with code 0
[Kredis-1 exited with code 0
backend-1-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: The message timed out in the backlog attempting to send because no connection became available (5000ms) - Last Connection Exception: It was not possible to connect to the redis server(s) redis:6379/Interactive. Error connecting right now. To allow this multiplexer to continue retrying until it's able to connect, use abortConnect=false in your connection string or AbortOnConnectFail=false; in your code. ConnectTimeout, command=LPOP, timeout: 5000, inst: 0, qu: 14, qs: 0, aw: False, bw: CheckingForTimeout, last-in: 0, cur-in: 0, sync-ops: 0, async-ops: 80555, serverEndpoint: redis:6379, conn-sec: n/a, aoc: 1, mc: 1/1/0, mgr: 10 of 10 available, clientName: backend-1(SE.Redis-v2.8.58.19568), IOCP: (Busy=0,Free=1000,Min=1,Max=1000), WORKER: (Busy=2,Free=32765,Min=1,Max=32767), POOL: (Threads=3,QueuedItems=2,CompletedItems=303531,Timers=8), v: 2.8.58.19568 (Please take a look at this article for some common client-side issues that can cause timeouts: https://stackexchange.github.io/StackExchange.Redis/Timeouts)
backend-1-1      |  ---> StackExchange.Redis.RedisConnectionException: It was not possible to connect to the redis server(s) redis:6379/Interactive. Error connecting right now. To allow this multiplexer to continue retrying until it's able to connect, use abortConnect=false in your connection string or AbortOnConnectFail=false; in your code. ConnectTimeout
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x154
backend-1-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: The message timed out in the backlog attempting to send because no connection became available (5000ms) - Last Connection Exception: It was not possible to connect to the redis server(s) redis:6379/Interactive. Error connecting right now. To allow this multiplexer to continue retrying until it's able to connect, use abortConnect=false in your connection string or AbortOnConnectFail=false; in your code. ConnectTimeout, command=LPOP, timeout: 5000, inst: 0, qu: 13, qs: 0, aw: False, bw: CheckingForTimeout, rs: NotStarted, ws: Initializing, last-in: 0, cur-in: 0, sync-ops: 0, async-ops: 80555, serverEndpoint: redis:6379, conn-sec: n/a, aoc: 1, mc: 1/1/0, mgr: 10 of 10 available, clientName: backend-1(SE.Redis-v2.8.58.19568), IOCP: (Busy=0,Free=1000,Min=1,Max=1000), WORKER: (Busy=2,Free=32765,Min=1,Max=32767), POOL: (Threads=3,QueuedItems=3,CompletedItems=303531,Timers=8), v: 2.8.58.19568 (Please take a look at this article for some common client-side issues that can cause timeouts: https://stackexchange.github.io/StackExchange.Redis/Timeouts)
backend-1-1      |  ---> StackExchange.Redis.RedisConnectionException: It was not possible to connect to the redis server(s) redis:6379/Interactive. Error connecting right now. To allow this multiplexer to continue retrying until it's able to connect, use abortConnect=false in your connection string or AbortOnConnectFail=false; in your code. ConnectTimeout
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x154
backend-1-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: The message timed out in the backlog attempting to send because no connection became available (5000ms) - Last Connection Exception: It was not possible to connect to the redis server(s) redis:6379/Interactive. Error connecting right now. To allow this multiplexer to continue retrying until it's able to connect, use abortConnect=false in your connection string or AbortOnConnectFail=false; in your code. ConnectTimeout, command=LPOP, timeout: 5000, inst: 0, qu: 12, qs: 0, aw: False, bw: CheckingForTimeout, rs: NotStarted, ws: Initializing, in: 0, last-in: 0, cur-in: 0, sync-ops: 0, async-ops: 80555, serverEndpoint: redis:6379, conn-sec: n/a, aoc: 1, mc: 1/1/0, mgr: 10 of 10 available, clientName: backend-1(SE.Redis-v2.8.58.19568), IOCP: (Busy=0,Free=1000,Min=1,Max=1000), WORKER: (Busy=2,Free=32765,Min=1,Max=32767), POOL: (Threads=3,QueuedItems=4,CompletedItems=303531,Timers=8), v: 2.8.58.19568 (Please take a look at this article for some common client-side issues that can cause timeouts: https://stackexchange.github.io/StackExchange.Redis/Timeouts)
backend-1-1      |  ---> StackExchange.Redis.RedisConnectionException: It was not possible to connect to the redis server(s) redis:6379/Interactive. Error connecting right now. To allow this multiplexer to continue retrying until it's able to connect, use abortConnect=false in your connection string or AbortOnConnectFail=false; in your code. ConnectTimeout
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x154
backend-1-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: The message timed out in the backlog attempting to send because no connection became available (5000ms) - Last Connection Exception: It was not possible to connect to the redis server(s) redis:6379/Interactive. Error connecting right now. To allow this multiplexer to continue retrying until it's able to connect, use abortConnect=false in your connection string or AbortOnConnectFail=false; in your code. ConnectTimeout, command=LPOP, timeout: 5000, inst: 0, qu: 11, qs: 0, aw: False, bw: CheckingForTimeout, rs: NotStarted, ws: Initializing, in: 0, last-in: 0, cur-in: 0, sync-ops: 0, async-ops: 80555, serverEndpoint: redis:6379, conn-sec: n/a, aoc: 1, mc: 1/1/0, mgr: 10 of 10 available, clientName: backend-1(SE.Redis-v2.8.58.19568), IOCP: (Busy=0,Free=1000,Min=1,Max=1000), WORKER: (Busy=2,Free=32765,Min=1,Max=32767), POOL: (Threads=3,QueuedItems=5,CompletedItems=303531,Timers=9), v: 2.8.58.19568 (Please take a look at this article for some common client-side issues that can cause timeouts: https://stackexchange.github.io/StackExchange.Redis/Timeouts)
backend-1-1      |  ---> StackExchange.Redis.RedisConnectionException: It was not possible to connect to the redis server(s) redis:6379/Interactive. Error connecting right now. To allow this multiplexer to continue retrying until it's able to connect, use abortConnect=false in your connection string or AbortOnConnectFail=false; in your code. ConnectTimeout
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x154
backend-1-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: The message timed out in the backlog attempting to send because no connection became available (5000ms) - Last Connection Exception: It was not possible to connect to the redis server(s) redis:6379/Interactive. Error connecting right now. To allow this multiplexer to continue retrying until it's able to connect, use abortConnect=false in your connection string or AbortOnConnectFail=false; in your code. ConnectTimeout, command=LPOP, timeout: 5000, inst: 0, qu: 10, qs: 0, aw: False, bw: CheckingForTimeout, rs: NotStarted, ws: Initializing, in: 0, last-in: 0, cur-in: 0, sync-ops: 0, async-ops: 80555, serverEndpoint: redis:6379, conn-sec: n/a, aoc: 1, mc: 1/1/0, mgr: 10 of 10 available, clientName: backend-1(SE.Redis-v2.8.58.19568), IOCP: (Busy=0,Free=1000,Min=1,Max=1000), WORKER: (Busy=2,Free=32765,Min=1,Max=32767), POOL: (Threads=3,QueuedItems=6,CompletedItems=303531,Timers=9), v: 2.8.58.19568 (Please take a look at this article for some common client-side issues that can cause timeouts: https://stackexchange.github.io/StackExchange.Redis/Timeouts)
backend-1-1      |  ---> StackExchange.Redis.RedisConnectionException: It was not possible to connect to the redis server(s) redis:6379/Interactive. Error connecting right now. To allow this multiplexer to continue retrying until it's able to connect, use abortConnect=false in your connection string or AbortOnConnectFail=false; in your code. ConnectTimeout
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x154
backend-1-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: The message timed out in the backlog attempting to send because no connection became available (5000ms) - Last Connection Exception: It was not possible to connect to the redis server(s) redis:6379/Interactive. Error connecting right now. To allow this multiplexer to continue retrying until it's able to connect, use abortConnect=false in your connection string or AbortOnConnectFail=false; in your code. ConnectTimeout, command=LPOP, timeout: 5000, inst: 0, qu: 9, qs: 0, aw: False, bw: CheckingForTimeout, rs: NotStarted, ws: Initializing, in: 0, last-in: 0, cur-in: 0, sync-ops: 0, async-ops: 80555, serverEndpoint: redis:6379, conn-sec: n/a, aoc: 1, mc: 1/1/0, mgr: 10 of 10 available, clientName: backend-1(SE.Redis-v2.8.58.19568), IOCP: (Busy=0,Free=1000,Min=1,Max=1000), WORKER: (Busy=2,Free=32765,Min=1,Max=32767), POOL: (Threads=3,QueuedItems=7,CompletedItems=303531,Timers=9), v: 2.8.58.19568 (Please take a look at this article for some common client-side issues that can cause timeouts: https://stackexchange.github.io/StackExchange.Redis/Timeouts)
backend-1-1      |  ---> StackExchange.Redis.RedisConnectionException: It was not possible to connect to the redis server(s) redis:6379/Interactive. Error connecting right now. To allow this multiplexer to continue retrying until it's able to connect, use abortConnect=false in your connection string or AbortOnConnectFail=false; in your code. ConnectTimeout
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x154
backend-1-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: The message timed out in the backlog attempting to send because no connection became available (5000ms) - Last Connection Exception: It was not possible to connect to the redis server(s) redis:6379/Interactive. Error connecting right now. To allow this multiplexer to continue retrying until it's able to connect, use abortConnect=false in your connection string or AbortOnConnectFail=false; in your code. ConnectTimeout, command=LPOP, timeout: 5000, inst: 0, qu: 8, qs: 0, aw: False, bw: CheckingForTimeout, rs: NotStarted, ws: Initializing, in: 0, last-in: 0, cur-in: 0, sync-ops: 0, async-ops: 80555, serverEndpoint: redis:6379, conn-sec: n/a, aoc: 1, mc: 1/1/0, mgr: 10 of 10 available, clientName: backend-1(SE.Redis-v2.8.58.19568), IOCP: (Busy=0,Free=1000,Min=1,Max=1000), WORKER: (Busy=2,Free=32765,Min=1,Max=32767), POOL: (Threads=3,QueuedItems=8,CompletedItems=303531,Timers=9), v: 2.8.58.19568 (Please take a look at this article for some common client-side issues that can cause timeouts: https://stackexchange.github.io/StackExchange.Redis/Timeouts)
backend-1-1      |  ---> StackExchange.Redis.RedisConnectionException: It was not possible to connect to the redis server(s) redis:6379/Interactive. Error connecting right now. To allow this multiplexer to continue retrying until it's able to connect, use abortConnect=false in your connection string or AbortOnConnectFail=false; in your code. ConnectTimeout
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x154
backend-1-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: The message timed out in the backlog attempting to send because no connection became available (5000ms) - Last Connection Exception: It was not possible to connect to the redis server(s) redis:6379/Interactive. Error connecting right now. To allow this multiplexer to continue retrying until it's able to connect, use abortConnect=false in your connection string or AbortOnConnectFail=false; in your code. ConnectTimeout, command=LPOP, timeout: 5000, inst: 0, qu: 7, qs: 0, aw: False, bw: CheckingForTimeout, rs: NotStarted, ws: Initializing, in: 0, last-in: 0, cur-in: 0, sync-ops: 0, async-ops: 80555, serverEndpoint: redis:6379, conn-sec: n/a, aoc: 1, mc: 1/1/0, mgr: 10 of 10 available, clientName: backend-1(SE.Redis-v2.8.58.19568), IOCP: (Busy=0,Free=1000,Min=1,Max=1000), WORKER: (Busy=2,Free=32765,Min=1,Max=32767), POOL: (Threads=3,QueuedItems=9,CompletedItems=303531,Timers=9), v: 2.8.58.19568 (Please take a look at this article for some common client-side issues that can cause timeouts: https://stackexchange.github.io/StackExchange.Redis/Timeouts)
backend-1-1      |  ---> StackExchange.Redis.RedisConnectionException: It was not possible to connect to the redis server(s) redis:6379/Interactive. Error connecting right now. To allow this multiplexer to continue retrying until it's able to connect, use abortConnect=false in your connection string or AbortOnConnectFail=false; in your code. ConnectTimeout
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x154
backend-1-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: The message timed out in the backlog attempting to send because no connection became available (5000ms) - Last Connection Exception: It was not possible to connect to the redis server(s) redis:6379/Interactive. Error connecting right now. To allow this multiplexer to continue retrying until it's able to connect, use abortConnect=false in your connection string or AbortOnConnectFail=false; in your code. ConnectTimeout, command=LPOP, timeout: 5000, inst: 0, qu: 6, qs: 0, aw: False, bw: CheckingForTimeout, rs: NotStarted, ws: Initializing, in: 0, last-in: 0, cur-in: 0, sync-ops: 0, async-ops: 80555, serverEndpoint: redis:6379, conn-sec: n/a, aoc: 1, mc: 1/1/0, mgr: 10 of 10 available, clientName: backend-1(SE.Redis-v2.8.58.19568), IOCP: (Busy=0,Free=1000,Min=1,Max=1000), WORKER: (Busy=2,Free=32765,Min=1,Max=32767), POOL: (Threads=3,QueuedItems=10,CompletedItems=303531,Timers=9), v: 2.8.58.19568 (Please take a look at this article for some common client-side issues that can cause timeouts: https://stackexchange.github.io/StackExchange.Redis/Timeouts)
backend-1-1      |  ---> StackExchange.Redis.RedisConnectionException: It was not possible to connect to the redis server(s) redis:6379/Interactive. Error connecting right now. To allow this multiplexer to continue retrying until it's able to connect, use abortConnect=false in your connection string or AbortOnConnectFail=false; in your code. ConnectTimeout
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x154
backend-1-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: The message timed out in the backlog attempting to send because no connection became available (5000ms) - Last Connection Exception: It was not possible to connect to the redis server(s) redis:6379/Interactive. Error connecting right now. To allow this multiplexer to continue retrying until it's able to connect, use abortConnect=false in your connection string or AbortOnConnectFail=false; in your code. ConnectTimeout, command=LPOP, timeout: 5000, inst: 0, qu: 5, qs: 0, aw: False, bw: CheckingForTimeout, rs: NotStarted, ws: Initializing, in: 0, last-in: 0, cur-in: 0, sync-ops: 0, async-ops: 80555, serverEndpoint: redis:6379, conn-sec: n/a, aoc: 1, mc: 1/1/0, mgr: 10 of 10 available, clientName: backend-1(SE.Redis-v2.8.58.19568), IOCP: (Busy=0,Free=1000,Min=1,Max=1000), WORKER: (Busy=2,Free=32765,Min=1,Max=32767), POOL: (Threads=3,QueuedItems=11,CompletedItems=303531,Timers=9), v: 2.8.58.19568 (Please take a look at this article for some common client-side issues that can cause timeouts: https://stackexchange.github.io/StackExchange.Redis/Timeouts)
backend-1-1      |  ---> StackExchange.Redis.RedisConnectionException: It was not possible to connect to the redis server(s) redis:6379/Interactive. Error connecting right now. To allow this multiplexer to continue retrying until it's able to connect, use abortConnect=false in your connection string or AbortOnConnectFail=false; in your code. ConnectTimeout
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x154
backend-1-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: The message timed out in the backlog attempting to send because no connection became available (5000ms) - Last Connection Exception: It was not possible to connect to the redis server(s) redis:6379/Interactive. Error connecting right now. To allow this multiplexer to continue retrying until it's able to connect, use abortConnect=false in your connection string or AbortOnConnectFail=false; in your code. ConnectTimeout, command=LPOP, timeout: 5000, inst: 0, qu: 4, qs: 0, aw: False, bw: CheckingForTimeout, rs: NotStarted, ws: Initializing, in: 0, last-in: 0, cur-in: 0, sync-ops: 0, async-ops: 80555, serverEndpoint: redis:6379, conn-sec: n/a, aoc: 1, mc: 1/1/0, mgr: 10 of 10 available, clientName: backend-1(SE.Redis-v2.8.58.19568), IOCP: (Busy=0,Free=1000,Min=1,Max=1000), WORKER: (Busy=2,Free=32765,Min=1,Max=32767), POOL: (Threads=3,QueuedItems=11,CompletedItems=303532,Timers=10), v: 2.8.58.19568 (Please take a look at this article for some common client-side issues that can cause timeouts: https://stackexchange.github.io/StackExchange.Redis/Timeouts)
backend-1-1      |  ---> StackExchange.Redis.RedisConnectionException: It was not possible to connect to the redis server(s) redis:6379/Interactive. Error connecting right now. To allow this multiplexer to continue retrying until it's able to connect, use abortConnect=false in your connection string or AbortOnConnectFail=false; in your code. ConnectTimeout
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x154
backend-1-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: The message timed out in the backlog attempting to send because no connection became available (5000ms) - Last Connection Exception: It was not possible to connect to the redis server(s) redis:6379/Interactive. Error connecting right now. To allow this multiplexer to continue retrying until it's able to connect, use abortConnect=false in your connection string or AbortOnConnectFail=false; in your code. ConnectTimeout, command=LPOP, timeout: 5000, inst: 0, qu: 3, qs: 0, aw: False, bw: CheckingForTimeout, rs: NotStarted, ws: Initializing, in: 0, last-in: 0, cur-in: 0, sync-ops: 0, async-ops: 80555, serverEndpoint: redis:6379, conn-sec: n/a, aoc: 1, mc: 1/1/0, mgr: 10 of 10 available, clientName: backend-1(SE.Redis-v2.8.58.19568), IOCP: (Busy=0,Free=1000,Min=1,Max=1000), WORKER: (Busy=2,Free=32765,Min=1,Max=32767), POOL: (Threads=3,QueuedItems=10,CompletedItems=303536,Timers=10), v: 2.8.58.19568 (Please take a look at this article for some common client-side issues that can cause timeouts: https://stackexchange.github.io/StackExchange.Redis/Timeouts)
backend-1-1      |  ---> StackExchange.Redis.RedisConnectionException: It was not possible to connect to the redis server(s) redis:6379/Interactive. Error connecting right now. To allow this multiplexer to continue retrying until it's able to connect, use abortConnect=false in your connection string or AbortOnConnectFail=false; in your code. ConnectTimeout
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x154
backend-1-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: The message timed out in the backlog attempting to send because no connection became available (5000ms) - Last Connection Exception: It was not possible to connect to the redis server(s) redis:6379/Interactive. Error connecting right now. To allow this multiplexer to continue retrying until it's able to connect, use abortConnect=false in your connection string or AbortOnConnectFail=false; in your code. ConnectTimeout, command=LPOP, timeout: 5000, inst: 0, qu: 2, qs: 0, aw: False, bw: CheckingForTimeout, rs: NotStarted, ws: Initializing, in: 0, last-in: 0, cur-in: 0, sync-ops: 0, async-ops: 80555, serverEndpoint: redis:6379, conn-sec: n/a, aoc: 1, mc: 1/1/0, mgr: 10 of 10 available, clientName: backend-1(SE.Redis-v2.8.58.19568), IOCP: (Busy=0,Free=1000,Min=1,Max=1000), WORKER: (Busy=2,Free=32765,Min=1,Max=32767), POOL: (Threads=3,QueuedItems=11,CompletedItems=303536,Timers=10), v: 2.8.58.19568 (Please take a look at this article for some common client-side issues that can cause timeouts: https://stackexchange.github.io/StackExchange.Redis/Timeouts)
backend-1-1      |  ---> StackExchange.Redis.RedisConnectionException: It was not possible to connect to the redis server(s) redis:6379/Interactive. Error connecting right now. To allow this multiplexer to continue retrying until it's able to connect, use abortConnect=false in your connection string or AbortOnConnectFail=false; in your code. ConnectTimeout
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x154
backend-1-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: The message timed out in the backlog attempting to send because no connection became available (5000ms) - Last Connection Exception: It was not possible to connect to the redis server(s) redis:6379/Interactive. Error connecting right now. To allow this multiplexer to continue retrying until it's able to connect, use abortConnect=false in your connection string or AbortOnConnectFail=false; in your code. ConnectTimeout, command=LPOP, timeout: 5000, inst: 0, qu: 1, qs: 0, aw: False, bw: CheckingForTimeout, rs: NotStarted, ws: Initializing, in: 0, last-in: 0, cur-in: 0, sync-ops: 0, async-ops: 80555, serverEndpoint: redis:6379, conn-sec: n/a, aoc: 1, mc: 1/1/0, mgr: 10 of 10 available, clientName: backend-1(SE.Redis-v2.8.58.19568), IOCP: (Busy=0,Free=1000,Min=1,Max=1000), WORKER: (Busy=2,Free=32765,Min=1,Max=32767), POOL: (Threads=3,QueuedItems=12,CompletedItems=303536,Timers=10), v: 2.8.58.19568 (Please take a look at this article for some common client-side issues that can cause timeouts: https://stackexchange.github.io/StackExchange.Redis/Timeouts)
backend-1-1      |  ---> StackExchange.Redis.RedisConnectionException: It was not possible to connect to the redis server(s) redis:6379/Interactive. Error connecting right now. To allow this multiplexer to continue retrying until it's able to connect, use abortConnect=false in your connection string or AbortOnConnectFail=false; in your code. ConnectTimeout
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x154
backend-1-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: The message timed out in the backlog attempting to send because no connection became available (5000ms) - Last Connection Exception: It was not possible to connect to the redis server(s) redis:6379/Interactive. Error connecting right now. To allow this multiplexer to continue retrying until it's able to connect, use abortConnect=false in your connection string or AbortOnConnectFail=false; in your code. ConnectTimeout, command=LPOP, timeout: 5000, inst: 0, qu: 0, qs: 0, aw: False, bw: CheckingForTimeout, rs: NotStarted, ws: Initializing, in: 0, last-in: 0, cur-in: 0, sync-ops: 0, async-ops: 80555, serverEndpoint: redis:6379, conn-sec: n/a, aoc: 1, mc: 1/1/0, mgr: 10 of 10 available, clientName: backend-1(SE.Redis-v2.8.58.19568), IOCP: (Busy=0,Free=1000,Min=1,Max=1000), WORKER: (Busy=2,Free=32765,Min=1,Max=32767), POOL: (Threads=3,QueuedItems=13,CompletedItems=303536,Timers=10), v: 2.8.58.19568 (Please take a look at this article for some common client-side issues that can cause timeouts: https://stackexchange.github.io/StackExchange.Redis/Timeouts)
backend-1-1      |  ---> StackExchange.Redis.RedisConnectionException: It was not possible to connect to the redis server(s) redis:6379/Interactive. Error connecting right now. To allow this multiplexer to continue retrying until it's able to connect, use abortConnect=false in your connection string or AbortOnConnectFail=false; in your code. ConnectTimeout
backend-1-1      |    --- End of inner exception stack trace ---
backend-1-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x154
backend-2-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: The message timed out in the backlog attempting to send because no connection became available (5000ms) - Last Connection Exception: It was not possible to connect to the redis server(s) redis:6379/Interactive. Error connecting right now. To allow this multiplexer to continue retrying until it's able to connect, use abortConnect=false in your connection string or AbortOnConnectFail=false; in your code. ConnectTimeout, command=LPOP, timeout: 5000, inst: 0, qu: 14, qs: 0, aw: False, bw: CheckingForTimeout, last-in: 0, cur-in: 0, sync-ops: 0, async-ops: 79717, serverEndpoint: redis:6379, conn-sec: n/a, aoc: 1, mc: 1/1/0, mgr: 10 of 10 available, clientName: backend-2(SE.Redis-v2.8.58.19568), IOCP: (Busy=0,Free=1000,Min=1,Max=1000), WORKER: (Busy=2,Free=32765,Min=1,Max=32767), POOL: (Threads=3,QueuedItems=0,CompletedItems=306495,Timers=8), v: 2.8.58.19568 (Please take a look at this article for some common client-side issues that can cause timeouts: https://stackexchange.github.io/StackExchange.Redis/Timeouts)
backend-2-1      |  ---> StackExchange.Redis.RedisConnectionException: It was not possible to connect to the redis server(s) redis:6379/Interactive. Error connecting right now. To allow this multiplexer to continue retrying until it's able to connect, use abortConnect=false in your connection string or AbortOnConnectFail=false; in your code. ConnectTimeout
backend-2-1      |    --- End of inner exception stack trace ---
backend-2-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x154
backend-2-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: The message timed out in the backlog attempting to send because no connection became available (5000ms) - Last Connection Exception: It was not possible to connect to the redis server(s) redis:6379/Interactive. Error connecting right now. To allow this multiplexer to continue retrying until it's able to connect, use abortConnect=false in your connection string or AbortOnConnectFail=false; in your code. ConnectTimeout, command=LPOP, timeout: 5000, inst: 0, qu: 13, qs: 0, aw: False, bw: CheckingForTimeout, rs: NotStarted, ws: Initializing, last-in: 0, cur-in: 0, sync-ops: 0, async-ops: 79717, serverEndpoint: redis:6379, conn-sec: n/a, aoc: 1, mc: 1/1/0, mgr: 10 of 10 available, clientName: backend-2(SE.Redis-v2.8.58.19568), IOCP: (Busy=0,Free=1000,Min=1,Max=1000), WORKER: (Busy=2,Free=32765,Min=1,Max=32767), POOL: (Threads=3,QueuedItems=1,CompletedItems=306495,Timers=8), v: 2.8.58.19568 (Please take a look at this article for some common client-side issues that can cause timeouts: https://stackexchange.github.io/StackExchange.Redis/Timeouts)
backend-2-1      |  ---> StackExchange.Redis.RedisConnectionException: It was not possible to connect to the redis server(s) redis:6379/Interactive. Error connecting right now. To allow this multiplexer to continue retrying until it's able to connect, use abortConnect=false in your connection string or AbortOnConnectFail=false; in your code. ConnectTimeout
backend-2-1      |    --- End of inner exception stack trace ---
backend-2-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x154
backend-2-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: The message timed out in the backlog attempting to send because no connection became available (5000ms) - Last Connection Exception: It was not possible to connect to the redis server(s) redis:6379/Interactive. Error connecting right now. To allow this multiplexer to continue retrying until it's able to connect, use abortConnect=false in your connection string or AbortOnConnectFail=false; in your code. ConnectTimeout, command=LPOP, timeout: 5000, inst: 0, qu: 12, qs: 0, aw: False, bw: CheckingForTimeout, rs: NotStarted, ws: Initializing, in: 0, last-in: 0, cur-in: 0, sync-ops: 0, async-ops: 79717, serverEndpoint: redis:6379, conn-sec: n/a, aoc: 1, mc: 1/1/0, mgr: 10 of 10 available, clientName: backend-2(SE.Redis-v2.8.58.19568), IOCP: (Busy=0,Free=1000,Min=1,Max=1000), WORKER: (Busy=2,Free=32765,Min=1,Max=32767), POOL: (Threads=3,QueuedItems=2,CompletedItems=306495,Timers=9), v: 2.8.58.19568 (Please take a look at this article for some common client-side issues that can cause timeouts: https://stackexchange.github.io/StackExchange.Redis/Timeouts)
backend-2-1      |  ---> StackExchange.Redis.RedisConnectionException: It was not possible to connect to the redis server(s) redis:6379/Interactive. Error connecting right now. To allow this multiplexer to continue retrying until it's able to connect, use abortConnect=false in your connection string or AbortOnConnectFail=false; in your code. ConnectTimeout
backend-2-1      |    --- End of inner exception stack trace ---
backend-2-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x154
backend-2-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: The message timed out in the backlog attempting to send because no connection became available (5000ms) - Last Connection Exception: It was not possible to connect to the redis server(s) redis:6379/Interactive. Error connecting right now. To allow this multiplexer to continue retrying until it's able to connect, use abortConnect=false in your connection string or AbortOnConnectFail=false; in your code. ConnectTimeout, command=LPOP, timeout: 5000, inst: 0, qu: 11, qs: 0, aw: False, bw: CheckingForTimeout, rs: NotStarted, ws: Initializing, in: 0, last-in: 0, cur-in: 0, sync-ops: 0, async-ops: 79717, serverEndpoint: redis:6379, conn-sec: n/a, aoc: 1, mc: 1/1/0, mgr: 10 of 10 available, clientName: backend-2(SE.Redis-v2.8.58.19568), IOCP: (Busy=0,Free=1000,Min=1,Max=1000), WORKER: (Busy=2,Free=32765,Min=1,Max=32767), POOL: (Threads=3,QueuedItems=3,CompletedItems=306495,Timers=9), v: 2.8.58.19568 (Please take a look at this article for some common client-side issues that can cause timeouts: https://stackexchange.github.io/StackExchange.Redis/Timeouts)
backend-2-1      |  ---> StackExchange.Redis.RedisConnectionException: It was not possible to connect to the redis server(s) redis:6379/Interactive. Error connecting right now. To allow this multiplexer to continue retrying until it's able to connect, use abortConnect=false in your connection string or AbortOnConnectFail=false; in your code. ConnectTimeout
backend-2-1      |    --- End of inner exception stack trace ---
backend-2-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x154
backend-2-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: The message timed out in the backlog attempting to send because no connection became available (5000ms) - Last Connection Exception: It was not possible to connect to the redis server(s) redis:6379/Interactive. Error connecting right now. To allow this multiplexer to continue retrying until it's able to connect, use abortConnect=false in your connection string or AbortOnConnectFail=false; in your code. ConnectTimeout, command=LPOP, timeout: 5000, inst: 0, qu: 10, qs: 0, aw: False, bw: CheckingForTimeout, rs: NotStarted, ws: Initializing, in: 0, last-in: 0, cur-in: 0, sync-ops: 0, async-ops: 79717, serverEndpoint: redis:6379, conn-sec: n/a, aoc: 1, mc: 1/1/0, mgr: 10 of 10 available, clientName: backend-2(SE.Redis-v2.8.58.19568), IOCP: (Busy=0,Free=1000,Min=1,Max=1000), WORKER: (Busy=2,Free=32765,Min=1,Max=32767), POOL: (Threads=3,QueuedItems=4,CompletedItems=306495,Timers=9), v: 2.8.58.19568 (Please take a look at this article for some common client-side issues that can cause timeouts: https://stackexchange.github.io/StackExchange.Redis/Timeouts)
backend-2-1      |  ---> StackExchange.Redis.RedisConnectionException: It was not possible to connect to the redis server(s) redis:6379/Interactive. Error connecting right now. To allow this multiplexer to continue retrying until it's able to connect, use abortConnect=false in your connection string or AbortOnConnectFail=false; in your code. ConnectTimeout
backend-2-1      |    --- End of inner exception stack trace ---
backend-2-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x154
backend-2-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: The message timed out in the backlog attempting to send because no connection became available (5000ms) - Last Connection Exception: It was not possible to connect to the redis server(s) redis:6379/Interactive. Error connecting right now. To allow this multiplexer to continue retrying until it's able to connect, use abortConnect=false in your connection string or AbortOnConnectFail=false; in your code. ConnectTimeout, command=LPOP, timeout: 5000, inst: 0, qu: 9, qs: 0, aw: False, bw: CheckingForTimeout, rs: NotStarted, ws: Initializing, in: 0, last-in: 0, cur-in: 0, sync-ops: 0, async-ops: 79717, serverEndpoint: redis:6379, conn-sec: n/a, aoc: 1, mc: 1/1/0, mgr: 10 of 10 available, clientName: backend-2(SE.Redis-v2.8.58.19568), IOCP: (Busy=0,Free=1000,Min=1,Max=1000), WORKER: (Busy=2,Free=32765,Min=1,Max=32767), POOL: (Threads=3,QueuedItems=5,CompletedItems=306495,Timers=9), v: 2.8.58.19568 (Please take a look at this article for some common client-side issues that can cause timeouts: https://stackexchange.github.io/StackExchange.Redis/Timeouts)
backend-2-1      |  ---> StackExchange.Redis.RedisConnectionException: It was not possible to connect to the redis server(s) redis:6379/Interactive. Error connecting right now. To allow this multiplexer to continue retrying until it's able to connect, use abortConnect=false in your connection string or AbortOnConnectFail=false; in your code. ConnectTimeout
backend-2-1      |    --- End of inner exception stack trace ---
backend-2-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x154
backend-2-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: The message timed out in the backlog attempting to send because no connection became available (5000ms) - Last Connection Exception: It was not possible to connect to the redis server(s) redis:6379/Interactive. Error connecting right now. To allow this multiplexer to continue retrying until it's able to connect, use abortConnect=false in your connection string or AbortOnConnectFail=false; in your code. ConnectTimeout, command=LPOP, timeout: 5000, inst: 0, qu: 8, qs: 0, aw: False, bw: CheckingForTimeout, rs: NotStarted, ws: Initializing, in: 0, last-in: 0, cur-in: 0, sync-ops: 0, async-ops: 79717, serverEndpoint: redis:6379, conn-sec: n/a, aoc: 1, mc: 1/1/0, mgr: 10 of 10 available, clientName: backend-2(SE.Redis-v2.8.58.19568), IOCP: (Busy=0,Free=1000,Min=1,Max=1000), WORKER: (Busy=2,Free=32765,Min=1,Max=32767), POOL: (Threads=3,QueuedItems=6,CompletedItems=306495,Timers=9), v: 2.8.58.19568 (Please take a look at this article for some common client-side issues that can cause timeouts: https://stackexchange.github.io/StackExchange.Redis/Timeouts)
backend-2-1      |  ---> StackExchange.Redis.RedisConnectionException: It was not possible to connect to the redis server(s) redis:6379/Interactive. Error connecting right now. To allow this multiplexer to continue retrying until it's able to connect, use abortConnect=false in your connection string or AbortOnConnectFail=false; in your code. ConnectTimeout
backend-2-1      |    --- End of inner exception stack trace ---
backend-2-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x154
backend-2-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: The message timed out in the backlog attempting to send because no connection became available (5000ms) - Last Connection Exception: It was not possible to connect to the redis server(s) redis:6379/Interactive. Error connecting right now. To allow this multiplexer to continue retrying until it's able to connect, use abortConnect=false in your connection string or AbortOnConnectFail=false; in your code. ConnectTimeout, command=LPOP, timeout: 5000, inst: 0, qu: 7, qs: 0, aw: False, bw: CheckingForTimeout, rs: NotStarted, ws: Initializing, in: 0, last-in: 0, cur-in: 0, sync-ops: 0, async-ops: 79717, serverEndpoint: redis:6379, conn-sec: n/a, aoc: 1, mc: 1/1/0, mgr: 10 of 10 available, clientName: backend-2(SE.Redis-v2.8.58.19568), IOCP: (Busy=0,Free=1000,Min=1,Max=1000), WORKER: (Busy=2,Free=32765,Min=1,Max=32767), POOL: (Threads=3,QueuedItems=7,CompletedItems=306495,Timers=9), v: 2.8.58.19568 (Please take a look at this article for some common client-side issues that can cause timeouts: https://stackexchange.github.io/StackExchange.Redis/Timeouts)
backend-2-1      |  ---> StackExchange.Redis.RedisConnectionException: It was not possible to connect to the redis server(s) redis:6379/Interactive. Error connecting right now. To allow this multiplexer to continue retrying until it's able to connect, use abortConnect=false in your connection string or AbortOnConnectFail=false; in your code. ConnectTimeout
backend-2-1      |    --- End of inner exception stack trace ---
backend-2-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x154
backend-2-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: The message timed out in the backlog attempting to send because no connection became available (5000ms) - Last Connection Exception: It was not possible to connect to the redis server(s) redis:6379/Interactive. Error connecting right now. To allow this multiplexer to continue retrying until it's able to connect, use abortConnect=false in your connection string or AbortOnConnectFail=false; in your code. ConnectTimeout, command=LPOP, timeout: 5000, inst: 0, qu: 6, qs: 0, aw: False, bw: CheckingForTimeout, rs: NotStarted, ws: Initializing, in: 0, last-in: 0, cur-in: 0, sync-ops: 0, async-ops: 79717, serverEndpoint: redis:6379, conn-sec: n/a, aoc: 1, mc: 1/1/0, mgr: 10 of 10 available, clientName: backend-2(SE.Redis-v2.8.58.19568), IOCP: (Busy=0,Free=1000,Min=1,Max=1000), WORKER: (Busy=2,Free=32765,Min=1,Max=32767), POOL: (Threads=3,QueuedItems=8,CompletedItems=306495,Timers=9), v: 2.8.58.19568 (Please take a look at this article for some common client-side issues that can cause timeouts: https://stackexchange.github.io/StackExchange.Redis/Timeouts)
backend-2-1      |  ---> StackExchange.Redis.RedisConnectionException: It was not possible to connect to the redis server(s) redis:6379/Interactive. Error connecting right now. To allow this multiplexer to continue retrying until it's able to connect, use abortConnect=false in your connection string or AbortOnConnectFail=false; in your code. ConnectTimeout
backend-2-1      |    --- End of inner exception stack trace ---
backend-2-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x154
backend-2-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: The message timed out in the backlog attempting to send because no connection became available (5000ms) - Last Connection Exception: It was not possible to connect to the redis server(s) redis:6379/Interactive. Error connecting right now. To allow this multiplexer to continue retrying until it's able to connect, use abortConnect=false in your connection string or AbortOnConnectFail=false; in your code. ConnectTimeout, command=LPOP, timeout: 5000, inst: 0, qu: 5, qs: 0, aw: False, bw: CheckingForTimeout, rs: NotStarted, ws: Initializing, in: 0, last-in: 0, cur-in: 0, sync-ops: 0, async-ops: 79717, serverEndpoint: redis:6379, conn-sec: n/a, aoc: 1, mc: 1/1/0, mgr: 10 of 10 available, clientName: backend-2(SE.Redis-v2.8.58.19568), IOCP: (Busy=0,Free=1000,Min=1,Max=1000), WORKER: (Busy=2,Free=32765,Min=1,Max=32767), POOL: (Threads=3,QueuedItems=9,CompletedItems=306495,Timers=9), v: 2.8.58.19568 (Please take a look at this article for some common client-side issues that can cause timeouts: https://stackexchange.github.io/StackExchange.Redis/Timeouts)
backend-2-1      |  ---> StackExchange.Redis.RedisConnectionException: It was not possible to connect to the redis server(s) redis:6379/Interactive. Error connecting right now. To allow this multiplexer to continue retrying until it's able to connect, use abortConnect=false in your connection string or AbortOnConnectFail=false; in your code. ConnectTimeout
backend-2-1      |    --- End of inner exception stack trace ---
backend-2-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x154
backend-2-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: The message timed out in the backlog attempting to send because no connection became available (5000ms) - Last Connection Exception: It was not possible to connect to the redis server(s) redis:6379/Interactive. Error connecting right now. To allow this multiplexer to continue retrying until it's able to connect, use abortConnect=false in your connection string or AbortOnConnectFail=false; in your code. ConnectTimeout, command=LPOP, timeout: 5000, inst: 0, qu: 4, qs: 0, aw: False, bw: CheckingForTimeout, rs: NotStarted, ws: Initializing, in: 0, last-in: 0, cur-in: 0, sync-ops: 0, async-ops: 79717, serverEndpoint: redis:6379, conn-sec: n/a, aoc: 1, mc: 1/1/0, mgr: 10 of 10 available, clientName: backend-2(SE.Redis-v2.8.58.19568), IOCP: (Busy=0,Free=1000,Min=1,Max=1000), WORKER: (Busy=2,Free=32765,Min=1,Max=32767), POOL: (Threads=3,QueuedItems=9,CompletedItems=306496,Timers=10), v: 2.8.58.19568 (Please take a look at this article for some common client-side issues that can cause timeouts: https://stackexchange.github.io/StackExchange.Redis/Timeouts)
backend-2-1      |  ---> StackExchange.Redis.RedisConnectionException: It was not possible to connect to the redis server(s) redis:6379/Interactive. Error connecting right now. To allow this multiplexer to continue retrying until it's able to connect, use abortConnect=false in your connection string or AbortOnConnectFail=false; in your code. ConnectTimeout
backend-2-1      |    --- End of inner exception stack trace ---
backend-2-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x154
backend-2-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: The message timed out in the backlog attempting to send because no connection became available (5000ms) - Last Connection Exception: It was not possible to connect to the redis server(s) redis:6379/Interactive. Error connecting right now. To allow this multiplexer to continue retrying until it's able to connect, use abortConnect=false in your connection string or AbortOnConnectFail=false; in your code. ConnectTimeout, command=LPOP, timeout: 5000, inst: 0, qu: 3, qs: 0, aw: False, bw: CheckingForTimeout, rs: NotStarted, ws: Initializing, in: 0, last-in: 0, cur-in: 0, sync-ops: 0, async-ops: 79717, serverEndpoint: redis:6379, conn-sec: n/a, aoc: 1, mc: 1/1/0, mgr: 10 of 10 available, clientName: backend-2(SE.Redis-v2.8.58.19568), IOCP: (Busy=0,Free=1000,Min=1,Max=1000), WORKER: (Busy=2,Free=32765,Min=1,Max=32767), POOL: (Threads=3,QueuedItems=10,CompletedItems=306496,Timers=10), v: 2.8.58.19568 (Please take a look at this article for some common client-side issues that can cause timeouts: https://stackexchange.github.io/StackExchange.Redis/Timeouts)
backend-2-1      |  ---> StackExchange.Redis.RedisConnectionException: It was not possible to connect to the redis server(s) redis:6379/Interactive. Error connecting right now. To allow this multiplexer to continue retrying until it's able to connect, use abortConnect=false in your connection string or AbortOnConnectFail=false; in your code. ConnectTimeout
backend-2-1      |    --- End of inner exception stack trace ---
backend-2-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x154
backend-2-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: The message timed out in the backlog attempting to send because no connection became available (5000ms) - Last Connection Exception: It was not possible to connect to the redis server(s) redis:6379/Interactive. Error connecting right now. To allow this multiplexer to continue retrying until it's able to connect, use abortConnect=false in your connection string or AbortOnConnectFail=false; in your code. ConnectTimeout, command=LPOP, timeout: 5000, inst: 0, qu: 2, qs: 0, aw: False, bw: CheckingForTimeout, rs: NotStarted, ws: Initializing, in: 0, last-in: 0, cur-in: 0, sync-ops: 0, async-ops: 79717, serverEndpoint: redis:6379, conn-sec: n/a, aoc: 1, mc: 1/1/0, mgr: 10 of 10 available, clientName: backend-2(SE.Redis-v2.8.58.19568), IOCP: (Busy=0,Free=1000,Min=1,Max=1000), WORKER: (Busy=2,Free=32765,Min=1,Max=32767), POOL: (Threads=3,QueuedItems=11,CompletedItems=306496,Timers=10), v: 2.8.58.19568 (Please take a look at this article for some common client-side issues that can cause timeouts: https://stackexchange.github.io/StackExchange.Redis/Timeouts)
backend-2-1      |  ---> StackExchange.Redis.RedisConnectionException: It was not possible to connect to the redis server(s) redis:6379/Interactive. Error connecting right now. To allow this multiplexer to continue retrying until it's able to connect, use abortConnect=false in your connection string or AbortOnConnectFail=false; in your code. ConnectTimeout
backend-2-1      |    --- End of inner exception stack trace ---
backend-2-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x154
backend-2-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: The message timed out in the backlog attempting to send because no connection became available (5000ms) - Last Connection Exception: It was not possible to connect to the redis server(s) redis:6379/Interactive. Error connecting right now. To allow this multiplexer to continue retrying until it's able to connect, use abortConnect=false in your connection string or AbortOnConnectFail=false; in your code. ConnectTimeout, command=LPOP, timeout: 5000, inst: 0, qu: 1, qs: 0, aw: False, bw: CheckingForTimeout, rs: NotStarted, ws: Initializing, in: 0, last-in: 0, cur-in: 0, sync-ops: 0, async-ops: 79717, serverEndpoint: redis:6379, conn-sec: n/a, aoc: 1, mc: 1/1/0, mgr: 10 of 10 available, clientName: backend-2(SE.Redis-v2.8.58.19568), IOCP: (Busy=0,Free=1000,Min=1,Max=1000), WORKER: (Busy=2,Free=32765,Min=1,Max=32767), POOL: (Threads=3,QueuedItems=12,CompletedItems=306496,Timers=10), v: 2.8.58.19568 (Please take a look at this article for some common client-side issues that can cause timeouts: https://stackexchange.github.io/StackExchange.Redis/Timeouts)
backend-2-1      |  ---> StackExchange.Redis.RedisConnectionException: It was not possible to connect to the redis server(s) redis:6379/Interactive. Error connecting right now. To allow this multiplexer to continue retrying until it's able to connect, use abortConnect=false in your connection string or AbortOnConnectFail=false; in your code. ConnectTimeout
backend-2-1      |    --- End of inner exception stack trace ---
backend-2-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x154
backend-2-1      | [Worker Error] StackExchange.Redis.RedisConnectionException: The message timed out in the backlog attempting to send because no connection became available (5000ms) - Last Connection Exception: It was not possible to connect to the redis server(s) redis:6379/Interactive. Error connecting right now. To allow this multiplexer to continue retrying until it's able to connect, use abortConnect=false in your connection string or AbortOnConnectFail=false; in your code. ConnectTimeout, command=LPOP, timeout: 5000, inst: 0, qu: 0, qs: 0, aw: False, bw: CheckingForTimeout, rs: NotStarted, ws: Initializing, in: 0, last-in: 0, cur-in: 0, sync-ops: 0, async-ops: 79717, serverEndpoint: redis:6379, conn-sec: n/a, aoc: 1, mc: 1/1/0, mgr: 10 of 10 available, clientName: backend-2(SE.Redis-v2.8.58.19568), IOCP: (Busy=0,Free=1000,Min=1,Max=1000), WORKER: (Busy=2,Free=32765,Min=1,Max=32767), POOL: (Threads=3,QueuedItems=13,CompletedItems=306496,Timers=10), v: 2.8.58.19568 (Please take a look at this article for some common client-side issues that can cause timeouts: https://stackexchange.github.io/StackExchange.Redis/Timeouts)
backend-2-1      |  ---> StackExchange.Redis.RedisConnectionException: It was not possible to connect to the redis server(s) redis:6379/Interactive. Error connecting right now. To allow this multiplexer to continue retrying until it's able to connect, use abortConnect=false in your connection string or AbortOnConnectFail=false; in your code. ConnectTimeout
backend-2-1      |    --- End of inner exception stack trace ---
backend-2-1      |    at RedisQueueWorker.<WorkerLoopAsync>d__11.MoveNext() + 0x154
[Kbackend-1-1 exited with code 137
log truncated at line 1000
